\documentclass[12pt]{article}
\usepackage{scrtime} % for \thistime (this package MUST be listed first!)
\usepackage[margin=0.75in]{geometry}
\usepackage{graphicx}
\usepackage{fancyhdr}
\usepackage{caption}
\usepackage{subcaption}
\usepackage{xspace}
%\usepackage{underscore}
\usepackage{pdfpages}
\usepackage{xcolor,colortbl}%for changing cell colour
\usepackage{longtable}
\usepackage{hyperref}
\usepackage{booktabs}
\usepackage{array}
\pagestyle{fancy}
\setlength{\headheight}{15.2pt}
\setlength{\headsep}{13 pt}
\setlength{\parindent}{28 pt}
\setlength{\parskip}{12 pt}
\pagestyle{fancyplain}
\usepackage[T1]{fontenc}
\usepackage{tikz-cd}
\usepackage{tikz}
\usepackage[normalem]{ulem} %to strikeout text
\usetikzlibrary{decorations.markings}
\usetikzlibrary{calc, arrows}
\usepackage{lscape} %to make the page landscape
\usepackage{color,amsmath,amssymb,amsthm,mathrsfs,amsfonts,dsfont}
\usepackage{indentfirst} % to indent the first paragraph
\rhead{\fancyplain{}{Thesis Update \today \hfill Daniella Lato}}
%\rhead{\fancyplain{}{Thesis Update April 8, 2019 \hfill Daniella Lato}}
\title{Sinorhizobium Update}
\author{Daniella Lato}
\date{\today}
\renewcommand\headrulewidth{0.5mm}
\newcommand{\cc}{\cellcolor{black!16}}
\newcommand{\s}{\textit{Sinorhizobium}\xspace}
\newcommand{\smel}{\textit{S.\,meliloti}\xspace}
\newcommand{\smed}{\textit{S.\,medicae}\xspace}
\newcommand{\sfred}{\textit{S.\,fredii}\xspace}
\newcommand{\ssah}{\textit{S.\,saheli}\xspace}
\newcommand{\ster}{\textit{S.\,terangae}\xspace}
\newcommand{\agro}{\textit{A.\,tumefaciens}\xspace}
\newcommand{\escoli}{\textit{Escherichia coli}\xspace}
\newcommand{\bur}{\textit{Burkholderia}\xspace}
\newcommand{\vib}{\textit{Vibrio}\xspace}
\newcommand{\sul}{\textit{Sulfolobus}\xspace}
\newcommand{\ent}{\textit{Enterobacteria}\xspace}
\newcommand{\p}{progressiveMauve\xspace}
\newcommand{\bas}{\textit{Bacillus subtilis}\xspace}
\newcommand{\strep}{\textit{Streptomyces}\xspace}
\newcommand{\bass}{\textit{B.\,subtilis}\xspace}
\newcommand{\ecol}{\textit{E.\,coli}\xspace}
\newcommand{\ecoli}{\textit{Escherichia coli}\xspace}
\newcommand{\tub}{\textit{Mycobacterium tuberculosis}\xspace}
\newcommand{\pa}{pSymA\xspace}
\newcommand{\pb}{pSymB\xspace}
\newcommand{\snat}{\textit{S.\,natalensis}\xspace}
\newcommand{\scoe}{\textit{S.\,coelicolor}\xspace}
\newcommand{\borrb}{\textit{Borrelia burgdorferi}\xspace}
\providecommand{\e}[1]{\ensuremath{\times 10^{#1}}}
\newcommand{\ch}{$\checkmark$}
\newcommand{\dn}{\textit{dN}\xspace}
\newcommand{\ds}{\textit{dS}\xspace}
\newcommand{\sven}{\textit{S.\,venezuelae}\xspace}
%\newcommand{\scoe}{\textit{S.\,coelicolor}\xspace}
\newcommand{\sliv}{\textit{S.\,lividans}\xspace}
\newcommand{\bor}{\textit{Bordetella}\xspace}
\newcommand{\xan}{\textit{Xanthomonas}\xspace}
\begin{document}


\underline{Subs Paper Things to Do:}
\begin{itemize}

%	\item Or get 1st, 2nd, 3rd codon pos log regs

%    \item \sout{why does sinoC have omega lin reg = 0 near and far from the origin?}
%    
 %   \item create new graphs for selection analysis
%    
	\item why are the lin reg of \dn, \ds and $\omega$ NS but the subs graphs are...explain!

	\item mol clock for my analysis?
	
	\item GC content? COG? where do these fit?
	
\end{itemize}

\underline{Inversions and Gene Expression Letter Things to Do:}
\begin{itemize}
%	\item \sout{get as much GEO data as possible}
%	\item \sout{find papers about inversions and expression}
%	
%	\item \sout{see how many inversions I can identify in these strains of \ecoli with gene expression data}
%	
%	\item \sout{read papers about inversions}
%	
%	
%	\item \sout{check if opposite strand in \p means an inversions (check visual matches the xmfa)}
%	
%	\item \sout{check if PARSNP and \p both identify the same inversions (check xmfa file)}
	\item \sout{create latex template for paper}
%	\item \sout{put notes from papers into doc}
%	\item \sout{use large PARSNP alignment to identify inversions}


	\item look up inversions and small RNA's paper Marie was talking about at Committee meeting
	\item \sout{write outline for letter}
	\item write Abstract
	\item \sout{write intro}
	\item\sout{ write methods}
	\item compile tables (supplementary)
	\item \sout{write results}
	\item write discussion
	\item write conclusion 
	\item do same ancestral/phylogenetic analysis that I did in the subs paper 
\end{itemize}

\underline{General Things to Do:}
\begin{itemize}
	\item summarize references 40 and 56 from Committee meeting report (Brian was asking)
	
	
\end{itemize}

\section*{Last Week}



\textbf{Substitutions Paper:}

\ch address reviewers comment on HGT and ori/ter gradient

\ch address reviewers comment on annotation in outlier bars (mobile elements)

\ch create new cover letter for subst paper

\ch do subst analysis with only extant taxa branches (PAML and genome position)

\ch other analysis on subst paper

\textbf{Inversions $+$ Gene Expression:}

\ch double check Queenie's final dataframes 

\ch double check new inversion combos with Queenie's new data frames

\ch finish writing results for inversions paper (minus DESeq)

\ch clarify tests used in $\uparrow$

\ch attempt DESeq on sample\_name $\sim$ treatment

\ch double check for HNS and inversions fig that ALL HNS binding sites are shown (not just ones in inversions)

\ch continue get Lang and Oshima data from PDF to csv formats

\ch do H-NS analysis on $\uparrow$

\ch final decision on inversion viz (caption that explains it well): Figure \ref{fig:inversion_viz}


\textbf{Dissertation:}

\ch Finished dissertation discussion on gene expression paper

\ch Finished 2/3 proposed studies for dissertation

\ch Finished prefaces for each chapter

\ch Finished overall conclusion

\bigskip

\textbf{Inversions + Gene Expression:}
I attempted to use the individual sample names as a component in the DESeq analysis (sample\_name $\sim$ treatment).
But it did not work. DESeq still complains that the matrix is not full rank and the column factors (sample\_name and treatment) are linear combinations of each other.
\textbf{I do not know what to do. I need to account for variation in expression between datasets (since they were all done in different labs at different times), while still being able to see what differences are due to the ``treatment'' $=$ inversions. I would really appreciate your help and insight into this.}
 
I finished getting the Lang and Oshima data from PDF into a csv and complete the H-NS correlation analysis on these (Table \ref{tab:HNS}).
Oshima is NS so it is sort of irrelevant.
The Lang data however has a positive correlation with all inversions but a negative correlation with significant inversions (inversions that had a significant difference in gene expression between inverted and non-inverted sequences within the inversion).
\textbf{This is curious, and maybe related to the small number of data points? Do you have any thoughts on this?}
%\textbf{DESeq Analysis:}
%As I mentioned to you briefly last week, I was having trouble performing DESeq because my ``matrix is not full rank''. 
%This means that the combination of levels in each of my experimental design columns (treatment (inversion/non-inversion), strain, experiment) are co-linear.
%This is because the strain and inversion/non-inversion combinations are similar.
%For example, DESeq can not tell the difference between the ATCC strain and the inverted treatment, because it impacts the same samples (all the ATCC samples).
%Therefore, DESeq can not say if the differential expression is due to the strain or the inversion.
%I took your advice and tried to make my input data as simple as possible, but I am still getting this error.
%The only way that I will not get this error is if I look at each column in my experimental design matrix separately. 
%i.e. $\sim$ treatment, $\sim$ strain, $\sim$ experiment.
%However, based on my preliminary exploration of the data, it seems as though the experiment is driving most of the differential expression (which makes sense because the raw data is coming from a number of experiments that although we tried to use control data, they still were done in different labs, at different time...etc).
%\textbf{I am concerned that by doing $\sim$ treatment, and not $\sim$ treatment + experiment (which is what you usually do to combat say batch or lane effects), I am not accounting for the variation in expression between experiments. Do you have any thoughts on how I can ensure that the expression differences I am seeing with $\sim$ treatment are due to treatment along and not confounded by experiment? }
% 

%\textbf{Summary of Inversion and Exp Results:}
%I have included all the results for this analysis (except the DESeq analysis) in Tables \ref{tab:HNS} - \ref{tab:log_reg_strain}.
%
%I performed the H-NS analysis (looking at Pearson correlation between H-NS and Inversion/significant inversion) on each of the H-NS datasets (Table \ref{tab:HNS}).
%It seems as though within the Higashi 2016 dataset, it does not seem to matter which criteria for H-NS binding I use, they all give me roughly the same answer.
%This dataset also have multiple criteria for the non-coding H-NS binding so I looked at each non-coding criteria and the coding criteria 1, and again found that there appears to be no difference in the results depending on what criteria you choose.
%However, when we look across datasets, Grainger 2006 and Ueda 2013 have no significant correlation between H-NS binding and the inverted/significant inversions.
%In my data (meaning H-NS binding sites within my data), there are only 10 genes where all the H-NS datasets have the same binding sites.
%This could be why there is a difference in significance?
%For this reason, I think that the results from all the datasets need to be included in this paper/supplement.
%\textbf{I am not sure if it makes sense to say that H-NS has a positive correlation between inverted sites compared to non-inverted sites if some of the datasets do not show this. Thoughts?}
%
%In Table \ref{tab:overall_inver_exp}, there is a clear correlation between inverted blocks or individual inverted sequences and a significant difference in gene expression.
%When there is a significant difference in gene expression between inverted and non-inverted sequences within a block, inverted sequences have higher expression than non-inverted sequences about 58\% of the time (Table \ref{tab:multi_taxa_inver}).
%Table \ref{tab:len_summary} shows a positive correlation between block length and blocks with a significant difference in expression between inverted and non-inverted sequences.
%However, I am just looking into this now and I am confused by this. I will get back to you on that.
%
%When looking at distance from the origin of replication, we see that using all genomic positions across all strains (where inversions can be present in different genomic positions), there is no correlation between distance from the origin of replication and blocks with significant differences in gene expression between inverted and non-inverted sequences (Table \ref{tab:pos_summary}).
%Table \ref{tab:log_reg} shows multiple logistic regressions on the various values for inversions and distance from the origin of replication.
%This table again is considering points from all strains.
%The placement of inversion (all inversions, including ones with significant and non-significant differences in gene expression) appears to be concentrated near the terminus.
%When looking at only inversions with significant difference in gene expression, we see that these are located closer to the origin of replication.
%
%Table \ref{tab:log_reg_strain} shows the logistic regression between the inversion category (for all inversions, including ones that did not have a sig difference in gene expression) and distance from the origin of replication in each strain.
%The ``rev comp'' column is when that particular strain is inverted. The ``inversion'' column is when at least one strain in that block is inverted.
%Only \ecol K-12 DH10B and ATCC have inverted sequences, so really the other strains values are irrelevant.
%I also think the ``inversion'' column is irrelevant because it is the same data for each strain just with different genomic positions.
%We can see that the ATCC strain has most of its inversions concentrated near the origin of replication and less near the terminus.
%In the K-12 DH10B  there was no significant correlation with distance from the origin of replication and placement of inversions.

%\textbf{Inversion Visualization:}
%I have figured out the differences between the images I presented you with last time (where ATCC was reverse complemented or not).
%Each time you run PARSNP, even if it is on the same sequences but one is reverse complemented, it (obviously) runs the entire algorithm again.
%However, the output creates subtle difference in the sequences within the blocks.
%So some blocks that did not have a rearrangement when ATCC was not reverse complemented, have a rearrangement when ATCC is reverse complemented.
%This is why there appears to be more rearrangements between BW25113 and K-12 DH10B in Figure \ref{fig:inversion_revcomp} (ATCC reverse complemented) compared to Figure \ref{fig:inversion_norevcomp} (ATCC not reverse complemented).
%I cheated a bit and did a ``manual'' reverse complement of the ATCC block locations from the original data (where ATCC is not reverse complemented) by simply reversing the genomic positions (not altering the sequence data or anything else at all).
%This resulted in a much ``cleaner'' picture of the inversions (Figure \ref{fig:inversion_manual_revcomp}).
%This only reverse complemented the inversions visually in ATCC and did not create new rearrangements between BW25113 and K-12 DH10B.
%\textbf{The reason I wanted to ``clean up'' the inversion visualization is to combat reviewers comments. I suspect a reviewer will say ``it looks like the entire ATCC genome is inverted (in Figure \ref{fig:inversion_norevcomp}), so why didn't you fix this? Are your inversions accurate?'' I am unsure if it is ``correct'' to use the manual reverse complement cleaned version (Figure \ref{fig:inversion_manual_revcomp}) and say that ATCC was reverse complemented to simplify the diagram? But then I think maybe the inversions should also be manually reversed (so anything that was an inversion when it was not reverse complemented would now not be an inversion, and vice versa). But again, I am not sure if this is ``correct''. What do you think about all this?}


%I also noticed from Figures \ref{fig:inversion_norevcomp} and \ref{fig:inversion_revcomp} that a lot of the inversions have been rearranged in addition to being inverted.
%I am not sure how best to account for this in my calculations.
%\textbf{Do you have any thoughts on this?}


\textbf{Subst Paper:}

I finished addressing all the other reviewers comments in the paper and in the cover letter (which I will send to you soon).
One of the comments was to check what types of genes were present in the outlier bars from the substitution figures, and perhaps if these are mobile elements, this may explain why these bars have high substitutions and are considered outliers.
I looked into this and almost all mobile elements or phage-like genes were removed either via annotation (no \texttt{mobile\_elements}) or due to poor alignment.
The reviewer suggested that I could look at the $\omega$ values of these mobile genes to see if they are what is driving the high substs in these bars.
\textbf{I was thinking, should I be looking at all $\omega$ values in these outlier bars? I was really only searching for mobile elements as the reviewer suggested. What do you think?}


As for the reviewer who is concerned about robust data, I was thinking about doing my analysis without accounting for rearrangements to show what those trends are (and hopefully they show a negative correlation like previous studies) and then show that when I account for rearrangements, then things change.
However, using \p inherently accounts for rearrangements so I am not sure if I should be re-aligning everything (and new trees...etc) with a different aligner that does not account for rearrangements?
But then I was thinking that you have to account for rearrangements somehow otherwise you would aligning non-homologous sequences.
So I looked into what some of the previous papers did (Cooper et. al 2010, Morrow et. al 2012...etc) and they all used basically the same protocol.
They identified orthologs via blastp (and often required these orthologs to have similar positions in the genome), aligned them with ClustalW, then used codeml to calculate rates.
\textbf{These studies also just chucked anything that had \ds $>$ 1 since these are unreliable. 
Should I be doing the same? This also got me thinking that I use different datasets for the subs analysis and the selection analysis. Should I be using the selection analysis to identify outlier genes and exclude those from my subst analysis??}

Anyways, this all got me thinking that perhaps my \p alignments are ok and I can just choose the extant branches from PAML to use in my logistic regression of subs v.s. distance from the origin of replication.
So I did this, where all the data and methods are the same except I only selected the tip branches and their corresponding substitutions and genomic positions.
The results are found in Table \ref{tab:sub_log_reg_no_ancest}, you can see that they are essentially the same as ``with rearrangements''.
However, based on my LOO we know that PAML can change what branch the substitutions are on based on the tree and what taxa are present.
\textbf{So maybe for this ``no rearrangements'' analysis I need to only count what substitutions we can see in the alignment, so use a program other than PAML? I am just concerned that PAML is moving this extant subs to deeper branches and then its not captured in this ``no rearrangements'' dataset. Thoughts?}

\textbf{Overall, I do not know what else to do about the robust-ness of the data to satisfy this reviewer. 
Unless you can think of anything else, I will just craft the cover letter addressing the reviewers concerns and saying that we have done all we can do.}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% HOW LARGE CAN THE SLOPE GET JUNE 1 2020
%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%I also did the test for looking at the substitutions slope and how large it can be.
%I did this by taking the largest non-outlier bar (weighted total substitutions/10Kbp) and set the position to 0, then making another fake point with position at the terminus and a substitutions value of 0, then computing a regression.
%The results can be found in Table \ref{tab:bigslope} and the actual slope values can be found in Table \ref{tab:cod_non_cod_log_reg}.
%The slopes still seem very small to me. \textbf{What do you think? How should I approach this accurately in the paper?}
%
%\begin{table}[h]
%	\centering
%	\resizebox{0.5\textwidth}{!}{%
%		\begin{tabular}{ll}
%			\toprule
%			Bacteria and Replicon &  Test Regression slope \\
%			\midrule
%			\ecol Chromosome & -2.94\e{-9} \\
%			\bass Chromosome &  -5.08\e{-9}\\
%			\strep Chromosome & -3.92\e{-10}\\
%			\smel Chromosome & -3.32\e{-10}\\
%			\smel pSymA & -5.66\e{-9} \\
%			\smel pSymB &  -5.65\e{-9}\\
%			\bottomrule
%		\end{tabular}
%		
%	}%resizebox
%	\caption{\label{tab:bigslope} Values of regression slope for each replicon using two points: 1) Highest weighted value of the number of substitutions / 10Kbp at position zero and 2) weighted value of the number of substitutions / 10Kbp of zero at the terminus. Simple linear regression was calculated. All results have no residuals (no residual degrees of freedom) because there are only two points on the line.}
%\end{table}



%I created a new theme for the selection and substitution graphs so that they all look relatively the same (similar margins, font size..etc).
%Last week when I was re-doing the SH-Test (to see which block trees were different from the overall tree), I realized that some of these blocks were not removed from the analysis in \ecol, \smel Chromosome and \pa.
%I spent most of this week re-running these analysis with the correct number of blocks.
%The new figures and results can be found below and in the attached Supplementary File for the paper. Nothing has changed significantly.
%\smel chromosome looks really terrible. All non-outlier points for \dn and $\omega$ are zero values. Therefore a regression is pointless and makes the selection graph look really odd.
%I started to look into this but it looks like the \smel chromosomes are just so similar that there are hardly any substitutions.
%This is particularly evident when you look at the backbone of the \p alignment (which shows similar sequences).
%When looking at the \ecoli alignement (Figure \ref{fig:ecoli_mauve}) we see that the backbone is very ``spiky'' indicating regions where the nucleotides are not similar.
%The same can be said for the \strep genomes (Figure \ref{fig:strep_mauve}), even though these are more similar to each other than the \ecoli genomes. 
%When we look at the \smel Chromosome alignment (Figure \ref{fig:sinoC_mauve}), we see that the backbone is almost completely flat, meaning that there is hardly any variation in the nucleotide sequence.
%This is especially curious because there appears to be no different in the average number of substitutions in the \smel chromosome (Table \ref{tab:avg_subs}).
%It could be that all the variation is being considered an ``outlier'' because most of the values are zero (because they are so smilar)?
%I am still really confused about why the selection graph for \smel Chromosome looks so odd and the only explanation I can come up with is that the sequences are just really really similar. \textbf{Do you have any thoughts on this or suggestions for other things I could investigate to figure out what is going on?}



\section*{This Week}
%aim to tick off 4 tasks a week
%
\begin{itemize}
	\item finish subst analysis with ``no rearrangements''
	\item write $\uparrow$ into polished cover letter and main paper
	\item submit subst paper
	\item actual analysis on DESeq data
	\item visualizations/results for $\uparrow$
	\item read papers on H-NS proteins
\item write inversions paper abstract

\end{itemize}


\section*{Next Week}
\begin{itemize}
\item read papers on H-NS proteins
\item write inversions paper discussion
\item write conclusion for dissertation
	\item maybe do inversions in 10kb blocks? (and other sliding windows?)
	\item dist from ori on DESeq results?
	\item HGT and HNS binding?
\end{itemize}

\newpage

%\begin{figure}
%	\includegraphics[width=\textwidth]{C:/Users/synch/Documents/PhD/inversion_and_gene_exp_work/Inversion_viz/parallel_sets_all.pdf}
%	\caption{\label{fig:inversion_norevcomp} Visualization of rearrangements and inversions in all \ecol strains. ATCC is in the GenBank listed orientation.}
%\end{figure}
%
%\begin{figure}
%	\includegraphics[width=\textwidth]{C:/Users/synch/Documents/PhD/inversion_and_gene_exp_work/Inversion_viz/parallel_sets_all_ATCC_revcomp.pdf}
%	\caption{\label{fig:inversion_revcomp} Visualization of rearrangements and inversions in all \ecol strains. ATCC is reverse complemented from the GenBank listed orientation.}
%\end{figure}
%
%\begin{figure}
%	\includegraphics[width=\textwidth]{C:/Users/synch/Documents/PhD/inversion_and_gene_exp_work/Inversion_viz/parallel_sets_maual_revcomp.pdf}
%	\caption{\label{fig:inversion_manual_revcomp} Visualization of rearrangements and inversions in all \ecol strains. ATCC is ``manually'' reverse complemented (only genomic position).}
%\end{figure}


\begin{table}[h]
	\centering
	\resizebox{0.5\textwidth}{!}{%
		\begin{tabular}{lc}
			\toprule
			Bacteria and Replicon & Protein Coding Sequences\\
			& Coefficient Estimate \\
			\midrule
			\ecol Chromosome & \cc -3.29\e{-8}*** \\
			\bass Chromosome & 8.70\e{-9}*\\
			\strep Chromosome & NS\\
			\smel Chromosome & \cc -6.80\e{-7}*** \\
			\smel pSymA & 4.49\e{-7}*** \\
			\smel pSymB & 6.27\e{-8}* \\
			\bottomrule
		\end{tabular}
		
	}%resizebox
	%	\caption{\label{tab:cod_non_cod_log_reg} Logistic regression analysis of the number of substitutions along all positions of the genome of the respective bacteria replicons. These genomic positions were split up into the coding and non-coding regions of the genome. Grey coloured boxes indicate a negative logistic regression coefficient estimate. All results are statistically significant. Logistic regression was calculated after the origin of replication was moved to the beginning of the genome and all subsequent positions were scaled around the origin accounting for bidirectional replication. All results are marked with significance codes as followed: $<$ 0.001 = `***', 0.001 $<$ 0.01 = `**', 0.01 $<$ 0.05 = `*', $>$ 0.05 = `NS'.}
	\caption{\label{tab:sub_log_reg_no_ancest} Logistic regression analysis of the number of substitutions along all protein coding positions of the genome of the respective bacteria replicons. ONLY EXTANT BRANCHES. Grey coloured boxes indicate a negative logistic regression coefficient estimate. All results are statistically significant. Logistic regression was calculated after the origin of replication was moved to the beginning of the genome and all subsequent positions were scaled around the origin accounting for bidirectional replication. All results are marked with significance codes as followed: $<$ 0.001 = `***', 0.001 $<$ 0.01 = `**', 0.01 $<$ 0.05 = `*', $>$ 0.05 = `NS'.}
\end{table}

%\begin{figure*}[t!]
%	\includegraphics[width=\textwidth]{C:/Users/synch/Documents/PhD/inversion_and_gene_exp_work/Inversion_viz/parallel_sets_all_colour.pdf}
%	\caption{\label{fig:inversion_viz} Visualization of rearrangements and inversions between all \ecol strains. The genome of each strain is represented as horizontal lines. Homologous alignment blocks in each taxa are connected with a winding vertical line. Alignment blocks can be found at varying genomic positions in each taxa. Inverted alignment blocks are coloured in dark purple and non-inverted alignment blocks (which include rearrangements) are coloured in light grey.}
%\end{figure*}

\begin{figure}
	\includegraphics[width=\textwidth]{C:/Users/synch/Documents/PhD/inversion_and_gene_exp_work/Stats/genome_pos_inversions_k12.pdf}
	\caption{\label{fig:inver_exp} Visualization of the difference in gene expression between inverted and non-inverted sequences within alignment blocks. Each alignment block represents homologous sequences between the \ecoli strains \textcolor{red}{insert table ref here}. \ecol K-12 MG1655 was used as the reference genome for genomic position for each alignment block. The midpoint of each alignment block was calculated to be the genomic distance from the \ecol K-12 MG1655 origin of replication.  Each alignment block has one point on the graph to represent the average expression value in \textbf{C}ounts \textbf{P}er \textbf{M}illion (\textbf{CPM}) for all inverted (circles) and non-inverted (triangles) sequences within the block. Blocks that had a significant difference in gene expression (using a Wilcoxon sign-ranked test, see Materials and Methods) have the inverted and non-inverted gene expression averages highlighted in pink circles and purple triangles respectively. A smoothing line (\texttt{loewss}) was added to link the average gene expression values for the inverted (pink solid) and non-inverted (purple dashed) sequences within block that had a significant difference in gene expression (using a Wilcoxon sign-ranked test, see Materials and Methods). All blocks that did not have a significant difference in average gene expression between inverted and non-inverted sequences within alignment blocks have the average inversion (circles) and non-inversion (triangles) gene expression values coloured in light grey.}
\end{figure}

\begin{figure}
	\includegraphics[width=\textwidth]{C:/Users/synch/Documents/PhD/inversion_and_gene_exp_work/Stats/hns_all_inversions.pdf}
	\caption{\label{fig:inver_hns} Visualization of the genomic locations of all inversion alignment blocks (light grey filled circles) identified between \ecol K-12 MG1655, \ecol K-12 DH10B, \ecol BW25113, and \ecol ATCC.
		The data points are plotted on the genome of \ecol K-12 MG1655 which is used as a reference. Each inversion alignment block has a single genomic location chosen to be the midpoint of the inverted region calculated to be the genomic distance from the \ecol K-12 MG1655 origin of replication.
		\textbf{H}istone-like \textbf{N}ucleoid-\textbf{S}tructuring (H-NS) protein binding sites in the \ecol K-12 MG1655 are overlaid on top of the inversion alignment blocks (circles outlined in dark purple). Data for the H-NS binding information is from Higashi \textcolor{red}{insert citation here.} Inversion alignment blocks that had a significant difference in gene expression between the inverted and non-inverted sequences within the block (using a Wilcoxon sign-ranked test, see Materials and Methods), are marked below the inverted alignment blocks with dark pink outlined triangles.}
\end{figure}


\begin{table}[h]
	\centering
	\resizebox{0.7\textwidth}{!}{%
		\begin{tabular}{lc}
			\toprule
			%			\cmidrule{8-10}
			%			& \multicolumn{3}{c}{Weighted} & \multicolumn{3}{c}{Non-weighted} & \multicolumn{3}{c}{Weighted} \\
			%			\cmidrule{2-3}
			%			\cmidrule{4-7}
			Strain Removed & Coefficient Estimate \\
			\midrule
			\multicolumn{2}{c}{\ecol} \\
			None & -2.66\e{-8}*** \\
			U00096 & -3.12\e{-8}***\\
			CP0032890 & -3.07\e{-8}*** \\
			CU9281640 & -2.95\e{-8}*** \\
			CP0018550 & -1.50\e{-8}*** \\
			BA0000070 & -2.63\e{-8}*** \\
			CU9281630 & -2.49\e{-8}***\\
			
			\midrule
			\multicolumn{2}{c}{\bass} \\
			None & 2.76\e{-8}***\\
			NC\_000964 & 2.96\e{-8}***\\
			NC\_018520 & 3.57\e{-8}***\\
			NC\_017195 & 1.00\e{-7}***\\
			NC\_022898 & 5.17\e{-8}*** \\
			NC\_014976 & \textcolor{red}{-4.02\e{-8}***} \\
			CP01731 & 5.43\e{-8}*** \\
			NC\_014479 & \textcolor{red}{NS}\\
			
			\midrule
			\multicolumn{2}{c}{\strep} \\
			None & 7.21\e{-8}***\\
			CP050522 & 8.37\e{-8}*** \\
			GG657756 & 3.62\e{-8}*** \\
			CP042324 & 7.72\e{-8}*** \\
			AL645882 & 7.65\e{-8}*** \\
			CM001889 & \textcolor{red}{-2.46\e{-7}***}\\
			
			\bottomrule
		\end{tabular}
		
	}%resizebox
	\caption{\label{tab:leave_one_out1} Logistic regression on the presence or absence of a substitution and distance from the origin of replication. Each strain was systematically removed and the entire analysis was repeated. All results are marked with significance codes as followed: $<$ 0.001 = `***', 0.001 $<$ 0.01 = `**', 0.01 $<$ 0.05 = `*', $>$ 0.05 = `NS'. }
\end{table}

\begin{table}[h]
	\centering
	\resizebox{0.7\textwidth}{!}{%
		\begin{tabular}{lc}
			\toprule
			%			\cmidrule{8-10}
			%			& \multicolumn{3}{c}{Weighted} & \multicolumn{3}{c}{Non-weighted} & \multicolumn{3}{c}{Weighted} \\
			%			\cmidrule{2-3}
			%			\cmidrule{4-7}
			Strain Removed & Coefficient Estimate \\
			\midrule
			\multicolumn{2}{c}{\smel Chromosome} \\
			None & -6.57\e{-7}***\\
			NC\_015590& -3.18\e{-7}***\\
			NC\_003047 & -6.01\e{-7}*** \\
			CP004140 & -6.00\e{-7}*** \\
			CP009144 & -6.67\e{-7}*** \\
			NC\_017322 & -7.19\e{-7}***\\
			NC\_017325 & -5.01\e{-7}***\\
			\midrule
			\multicolumn{2}{c}{\smel \pa} \\
			None & 2.74\e{-7}***\\
			NC\_017327 & 6.98\e{-7}***\\
			CP009145 & 1.78\e{-7}***\\
			NC\_003037 & 2.09\e{-7}***\\
			CP004138 & 2.08\e{-7}***\\
			NC\_015591 & \textcolor{red}{NS} \\
			NC\_017324 & \textcolor{red}{-1.52\e{-6}***} \\
			\midrule
			\multicolumn{2}{c}{\smel \pb} \\
			None & 1.10\e{-7}***\\
			NC\_015596 & 6.78\e{-7}***\\
			NC\_017326 & 1.67\e{-7}***\\
			NC\_017323 & \textcolor{red}{NS}\\
			CP009146 & \textcolor{red}{-2.57\e{-7}***}\\
			CP004139 & 1.04\e{-7}***\\
			NC\_003078 & 1.04\e{-7}*** \\
			\bottomrule
		\end{tabular}
		
	}%resizebox
	\caption{\label{tab:leave_one_out2} Logistic regression on the presence or absence of a substitution and distance from the origin of replication. Each strain was systematically removed and the entire analysis was repeated. All results are marked with significance codes as followed: $<$ 0.001 = `***', 0.001 $<$ 0.01 = `**', 0.01 $<$ 0.05 = `*', $>$ 0.05 = `NS'.}
\end{table}






\begin{table}[h]
	\centering
	\resizebox{\textwidth}{!}{%
		\begin{tabular}{lccc}
			\toprule
			H-NS Binding Study & All Inversions  & Significant Inversions & Total Number of\\
			&  H-NS Binding & and H-NS Binding  & H-NS Binding Sites\\
			& & & Within All Alignment Blocks \\
			\midrule
			Grainger 2006 \cite{Grai:06} & NS & NS & 54\\
			Ueda 2013 \cite{Ueda:13} & NS & NS & 276\\
			Higashi 2016 \cite{Higa:16} & &&\\
			\hspace*{10mm} criteria A & 0.0470* & 01135*** & 372\\
			\hspace*{10mm} criteria B & 0.0543** & 0.1034*** & 344\\
			\hspace*{10mm} criteria C & 0.0543** & 0.1034*** & 344\\
			\hspace*{10mm} criteria D & 0.0543** & 0.1034*** & 344\\
			\hspace*{10mm} criteria E & 0.0547** & 0.1042*** & 341\\
			\hspace*{10mm} criteria F & 0.0547**  & 0.1042*** & 341 \\
			Lang 2007 \cite{Lang:07} & 0.0572** & -0.0557* & 115\\ %composite data
			Oshima 2006 \cite{Oshi:06} & 0.0401* & NS & 667\\
			\bottomrule
		\end{tabular}
		
	}%resizebox
	\caption{\label{tab:HNS} \textcolor{red}{are there any other stats related to correlation that people like to have in these tables that I should also be including? }Pearson correlation between H-NS binding sites and inverted regions of the \ecol K-12 MG1655 genome. A genomic region was considered inverted if this sequence was inverted in any of the following four taxa: \ecol K-12 MG1655, \ecol K-12 DH10B, \ecol BW25113, and \ecol ATCC. The genomic positions of these inversions in \ecol K-12 MG1655 was used for reference. The binding sites for the H-NS protein are in the genomic coordinates of \ecol K-12 MG1655, chosen as a reference. The second column ``All Inversions and H-NS Binding'' represents the correlation coefficient between inverted regions and H-NS binding sites. The third column ``Significant Inversions and H-NS Binding'' represents the correlation coefficient between inverted regions with significant differences in normalized gene expression between inverted and non-inverted taxa (via a Wilcoxon signed-rank test) and H-NS binding sites. The \textcolor{red}{ref Higashi} data set had multiple criteria to define H-NS binding sites. They are listed as follows: A: Genes whose coding regions overlap with the H-NS binding regions, B: Genes whose coding regions overlap with the H-NS binding regions and intergenic regions that were bound by H-NS, C: Genes whose coding regions overlap with the H-NS binding regions and intergenic regions that are "class I " (see \textcolor{red}{cite Higashi}), D: Genes whose coding regions overlap with the H-NS binding regions and intergenic regions that contain known promoter sequences, E: Same as A, but genes on which H-NS binding is restricted to the 3' end and the length overlapping with H-NS-bound regions is <10\% of the total gene length were excluded from H-NS-bound genes, F: When genes included in transcriptional units whose upstream regions or first coding regions overlapped with H-NS bound regions, all genes in the transcriptional units were judged as genes affected by H-NS binding. All results are marked with significance codes as followed: $<$ 0.001 = `***', 0.001 $<$ 0.01 = `**', 0.01 $<$ 0.05 = `*', $>$ 0.05 = `NS'.}
\end{table}

\begin{table}[t!]
	\centering
	\resizebox{\textwidth}{!}{%
		\begin{tabular}{lc}
			\toprule
			Datasets:  & Correlation Coefficient (W)\\
			\midrule
			Inverted Blocks & 15218699**\\
			Inverted Sequences & 11436344***\\
			\bottomrule
		\end{tabular}
	}%resizebox
	\caption{\label{tab:overall_inver_exp} Correlation coefficients for Wilcoxon signed-rank test on various datasets to determine the correlation between an inversion and difference in normalized gene expression. The ``Inverted Blocks'' dataset represents alignment blocks that have at least one taxa with an inverted sequence. The ``Inverted Sequences'' dataset represents all individual sequences from all alignment blocks that were inverted. The correlation between both datasets was computed using a Wilcoxon signed-rank test. All results are marked with significance codes as followed: $<$ 0.001 = `***', 0.001 $<$ 0.01 = `**', 0.01 $<$ 0.05 = `*', $>$ 0.05 = `NS'.}
\end{table}


\begin{table}[t!]
	\centering
	\resizebox{\textwidth}{!}{%
		\begin{tabular}{ccc}
			\toprule
			 \multicolumn{3}{c}{\% of Blocks that are}  \\
			\cmidrule{1-3}
			Inverted & Inverted with &Increased in \\
			  & Differences in&Gene Expression\\
			  & Gene Expression & in Inverted Sequences \\
			\midrule
			68.29 & 8.22& 58.06 \\
			\bottomrule
		\end{tabular}
	}%resizebox
	\caption{\label{tab:multi_taxa_inver} Percent of blocks in categories for various datasets (blocks with all 4 taxa, at least 3 taxa, or at least 2 taxa). The second column is any block that had at least one sequences that was inverted. The last column only deals with blocks that had at least one inverted sequence and had a significant difference in gene expression (column 3).}
\end{table}

\begin{table}[t!]
	\centering
	\resizebox{\textwidth}{!}{%
		\begin{tabular}{c}
			\toprule
			Block Length Correlation Coefficient (W)\\
			\midrule
			4060729.5***\\
			\bottomrule
		\end{tabular}
	}%resizebox
	\caption{\label{tab:len_summary} Correlation coefficients for Wilcoxon signed-rank test in alignment blocks. The correlation coefficient represents a correlation between alignment block length and blocks with a significant/non-significant difference in normalized gene expression between inverted and non-inverted sequences within the block. All results are marked with significance codes as followed: $<$ 0.001 = `***', 0.001 $<$ 0.01 = `**', 0.01 $<$ 0.05 = `*', $>$ 0.05 = `NS'.}
\end{table}




\begin{table}[t!]
	\centering
	\resizebox{\textwidth}{!}{%
		\begin{tabular}{c}
			\toprule
			 Genomic Position Correlation Coefficient (W)\\
			\midrule
			 NS\\
			\bottomrule
		\end{tabular}
	}%resizebox
	\caption{\label{tab:pos_summary} Correlation coefficients for Wilcoxon signed-rank test in alignment blocks with a significant difference in normalized gene expression between inverted and non-inverted sequences within the block. The correlation coefficient between the significant blocks and the genomic position of the alignment blocks. All results are marked with significance codes as followed: $<$ 0.001 = `***', 0.001 $<$ 0.01 = `**', 0.01 $<$ 0.05 = `*', $>$ 0.05 = `NS'.}
\end{table}

\begin{table}[h]
	\centering
	\resizebox{\textwidth}{!}{%
		\begin{tabular}{lcc}
			\toprule
			Inversion Category & Correlation Coefficient \\
			\midrule
			rev comp & NS\\
			inversion & 2.20\e{-7}*** \\
			sig rev comp & -1.89\e{-7}*\\
			sig $\sim$ midpoint all blocks & NS \\
			sig $\sim$ midpoint inverted blocks & NS \\
% below category makes no sense bc all sig blocks will be labeled as inverted.
%			sig inversion & NS & NS\\
			\bottomrule
		\end{tabular}
		
	}%resizebox
	\caption{\label{tab:log_reg} Logistic regression between various inversion categories and distance from the origin of replication for all strains. rev comp = individual sequences inverted, inversion = block that has at least one inverted sequence, midpoint = block midpoint, sig = blocks with significant difference in normalized gene expression between inverted and non-inverted sequences within the block. All results are marked with significance codes as followed: $<$ 0.001 = `***', 0.001 $<$ 0.01 = `**', 0.01 $<$ 0.05 = `*', $>$ 0.05 = `NS'.}
\end{table}

\begin{table}[h]
	\centering
	\resizebox{\textwidth}{!}{%
		\begin{tabular}{lcc}
			\toprule
			Strain & rev comp & inversion \\
			\midrule
			%K12 and BW have no inverted seqs so they have no values for first col
			%no strains have values for second col because all sig blocks have inversion=1 for all rows
			\ecol K-12 MG1655 &  & 3.55\e{-7}*** \\
			\ecol K-12 DH10B & NS & 3.45\e{-7}*** \\
			\ecol BW25113 & & 3.73\e{-7}***\\
			\ecol ATCC & -1.92\e{-7}*** & -1.92\e{-7}*** \\
			\bottomrule
		\end{tabular}
		
	}%resizebox
	\caption{\label{tab:log_reg_strain} Logistic regression between various inversion categories and distance from the origin of replication for  each strain. rev comp = individual sequences inverted, inversion = block that has at least one inverted sequence, sig = blocks with significant difference in normalized gene expression between inverted and non-inverted sequences within the block. All results are marked with significance codes as followed: $<$ 0.001 = `***', 0.001 $<$ 0.01 = `**', 0.01 $<$ 0.05 = `*', $>$ 0.05 = `NS'.}
\end{table}


\end{document}
