\documentclass[12pt]{article}
\usepackage{scrtime} % for \thistime (this package MUST be listed first!)
\usepackage[margin=0.75in]{geometry}
\usepackage{graphicx}
\usepackage{fancyhdr}
\usepackage{xspace}
%\usepackage{underscore}
\usepackage{pdfpages}
\usepackage{xcolor,colortbl}%for changing cell colour
\usepackage{longtable}
\usepackage{booktabs}
\pagestyle{fancy}
\setlength{\headheight}{15.2pt}
\setlength{\headsep}{13 pt}
\setlength{\parindent}{28 pt}
\setlength{\parskip}{12 pt}
\pagestyle{fancyplain}
\usepackage[T1]{fontenc}
\usepackage{tikz-cd}
\usepackage{tikz}
\usepackage[normalem]{ulem} %to strikeout text
\usetikzlibrary{decorations.markings}
\usetikzlibrary{calc, arrows}
\usepackage{lscape} %to make the page landscape
\usepackage{color,amsmath,amssymb,amsthm,mathrsfs,amsfonts,dsfont}
\usepackage{indentfirst} % to indent the first paragraph
\rhead{\fancyplain{}{Thesis Update \today \hfill Daniella Lato}}
%\rhead{\fancyplain{}{Thesis Update April 8, 2019 \hfill Daniella Lato}}
\title{Sinorhizobium Update}
\author{Daniella Lato}
\date{\today}
\renewcommand\headrulewidth{0.5mm}
\newcommand{\cc}{\cellcolor{black!16}}
\newcommand{\s}{\textit{Sinorhizobium}\xspace}
\newcommand{\smel}{\textit{S.\,meliloti}\xspace}
\newcommand{\smed}{\textit{S.\,medicae}\xspace}
\newcommand{\sfred}{\textit{S.\,fredii}\xspace}
\newcommand{\ssah}{\textit{S.\,saheli}\xspace}
\newcommand{\ster}{\textit{S.\,terangae}\xspace}
\newcommand{\agro}{\textit{A.\,tumefaciens}\xspace}
\newcommand{\escoli}{\textit{Escherichia coli}\xspace}
\newcommand{\bur}{\textit{Burkholderia}\xspace}
\newcommand{\vib}{\textit{Vibrio}\xspace}
\newcommand{\sul}{\textit{Sulfolobus}\xspace}
\newcommand{\ent}{\textit{Enterobacteria}\xspace}
\newcommand{\p}{progressiveMauve\xspace}
\newcommand{\bas}{\textit{Bacillus subtilis}\xspace}
\newcommand{\strep}{\textit{Streptomyces}\xspace}
\newcommand{\bass}{\textit{B.\,subtilis}\xspace}
\newcommand{\ecol}{\textit{E.\,coli}\xspace}
\newcommand{\ecoli}{\textit{Escherichia coli}\xspace}
\newcommand{\tub}{\textit{Mycobacterium tuberculosis}\xspace}
\newcommand{\pa}{pSymA\xspace}
\newcommand{\pb}{pSymB\xspace}
\newcommand{\snat}{\textit{S.\,natalensis}\xspace}
\newcommand{\scoe}{\textit{S.\,coelicolor}\xspace}
\providecommand{\e}[1]{\ensuremath{\times 10^{#1}}}
\newcommand{\ch}{$\checkmark$}
\begin{document}
%	Nov 30:	Create graphs with slopes for each COG
	
	
%	Dec 3:	Create new binned scatter plot of COG log reg
	
%	Dec 6:	Determine if there are any other stats I want/need for COG stuff
	
%	Dec 10:	Calculate above stats and write in table
	
%	Dec 21:	Find papers on COG stuff (for intro and discussion), and other mol trends (discussion for sub paper)
	
%	Jan 6:	Read above mentioned papers and make notes
%	Oct 31: Write out methods for gene expression paper
%	
%	Sep 9: Think about/compile list of inversions in \ecol for new paper
%	
%	Nov 15: Think about how to better look at the COG data
%	
%	Nov 25: Complete any extra analysis needed for Substitution paper
%	
%	Dec 4: Mac Scholarships and Awards Due
%	
%	Dec 1: Write out COG methods
%	
%	Dec 15: Gather papers for COG paper intro
%	
%	Dec 15: Implement COG stuff
%	
%	Other things to do:
%	
%	Create outline for gene expression paper
%
%  research mito increased subs near origin
%
%  add above to subs paper writeup
%	
%	Write Write Write gene expression paper
%	
%	 Have gene expression/inversion data combined and in graphical format/regression lines calculated
%	
%	re-do gene exp/sub graphs with patchwork R package so they line up exactly
%	
%	Have data for other molecular trends (GC content, number of genes, essential gene lists..etc.) combined with graphs (or in supplement) for sub analysis
%
% organize all the notes I made for comps into topics that can be integrated into an intro if needed
%	
%	May 31:	Complete COG analysis
%	
%	Jun 30:	COG analysis Paper draft completed
%	
%	Jul 31: Add other mol trends to Sub Paper

\underline{Subs Paper Things to Do:}
\begin{itemize}
	\item \sout{ \# of coding and non-coding sites}
	
	\item \sout{\# of subs in each of $\uparrow$}
	
	\item \sout{Look into \strep non-coding issue}
	
	\item \sout{Look into \ecol coding issue}
	
	\item \sout{Look into \pb coding/non-coding trend weirdness}
	
	\item \sout{Figure out why \strep appears to have tons of coding data missing}
	
	\item \sout{Figure out what is going on with cod/non-cod code and why it is still not working!}
	
	\item \sout{write up methods for coding/non-coding}
	
	\item \sout{write methods and results for clustering}
	
	\item \sout{start code to split alignment into multiple alignments of each gene}
	
	\item \sout{figure out how to deal with overlapping genes}
	
	\item \sout{figure out how to deal with gaps in gene of ref taxa}
	
	\item \sout{split up the alignment into multiple alignments of each gene}
	
	\item \sout{check if each gene alignment is a multiple of 3 (proper codon alignment)}
	
	\item \sout{get dN/dS for coding/non-coding stuff per gene}
	 
	\item Or get 1st, 2nd, 3rd codon pos log regs
	
	\item \sout{write up coding/non-coding results}
	
	\item \sout{take out gene expression from this paper}
	
	\item \sout{write better intro/methods for distribution of subs graphs}
	
	\item \sout{write discussion for coding/non-coding}
	
	\item \sout{write coding/non-coding into conclusion}
	
	\item \sout{figured out pipeline for CODEML to calculate dN/dS for each gene}
	
	\item \sout{make a list of what should be in supplementary files for subs paper}
	
	\item \sout{put everything in list into supplementary file for subs paper}
	
	\item write dN/dS methods
	
	\item write dN/dS results
	
	\item write dN/dS discussion
	
	\item write dN/dS into conclusion
	
	\item \sout{new bar graph with coding and non-coding sites separated}
	
	\item mol clock for my analysis?
	
	\item GC content? COG? where do these fit?
	
\end{itemize}

\underline{Gene Expression Paper Things to Do:}
\begin{itemize}
	
	\item \sout{look for more GEO expression data for \smel}
	
	\item \sout{look for more GEO expression data for \strep}
	
	\item \sout{look for more GEO expression data for \bass}
	
	\item \sout{format paper and put in stuff that is already written}
	
	\item \sout{look for more GEO expression data for \ecol}
	
	\item \sout{Get numbers for how many different strains and multiples of each strain I have for gene expression}
	
	\item \sout{re-do gene expression analysis for \bass}
	
	\item \sout{re-do gene expression analysis for \ecol}
	
	\item \sout{find papers about what has been done with gene expression}
	
	\item \sout{read papers $\uparrow$}
	
	\item \sout{put notes from $\uparrow$ papers into word doc}
	
	\item write abstract
	
	\item \sout{write intro}
	
	\item add stuff from outline to Data section
	
	\item create graphs for expression distribution (no sub data)
	
	\item add \# of genes to expression graphs (top)
	
	\item average gene expression
	
	\item \sout{write discussion}
	
	\item write conclusion
	
	\item add into methods: filters for Hiseq, RT PCR and growth phases for data collection
	
	\item update supplementary figures/file
	
\end{itemize}

\underline{Inversions and Gene Expression Letter Things to Do:}
\begin{itemize}
	\item \sout{get as much GEO data as possible}
	\item \sout{find papers about inversions and expression}
	
	\item \sout{see how many inversions I can identify in these strains of \ecoli with gene expression data}
	
	\item \sout{read papers about inversions}
	
	
	\item \sout{check if opposite strand in \p means an inversions (check visual matches the xmfa)}
	
	\item \sout{check if PARSNP and \p both identify the same inversions (check xmfa file)}
	\item create latex template for paper
	\item \sout{put notes from papers into doc}
	\item use large PARSNP alignment to identify inversions
	\item confirm inversions with dot plot
	\item write outline for letter
	\item write Abstract
	\item write intro
	\item write methods
	\item compile tables (supplementary)
	\item write results
	\item write discussion
	\item write conclusion 
	\item do same ancestral/phylogenetic analysis that I did in the subs paper 
\end{itemize}


% next week look into how to calculate the dN/dS for the subs paper
% week of Dec 17th do same ancestral analysis on gene exp data for ecoli...which is going to require me to do everything from scratch so make a tree and all that jazz

	
\section*{Last Week}

\ch Finished re-running the coding and non-coding substitution analysis (Table \ref{tab:cod_non_cod_log_reg})

\ch Get per gene and per genome dN, dS, and $\omega$ results (Table \ref{tab:dN_dS_ratios})

\ch started thinking more about inversions and gene expression

\ch check if opposite strand in \p means an inversions (check visual matches the xmfa)

\ch check if PARSNP and \p both identify the same inversions (check xmfa file)

I re-ran the coding and non-coding substitution analysis because I realized that my genome position numbering was off for the blocks. The results are still the same and can be found in Table \ref{tab:cod_non_cod_log_reg}. The updated distribution of substitutions histograms are also below.

\textbf{dN, dS, $\omega$}
Calculated the per gene and per genome dN, dS, and $\omega$ results for each bacteria. For the per gen rates I calculated both a weighted average (weighted by the length of the gene) and the non-weighted average to see if there was a big difference.
In doing this, I noticed that my calculations for the per genome and weighted per gene averages were the same... So I need to look into this more to see what is happening.

I was also thinking about maybe associating each of the dN, dS, and $\omega$ averages per gene with the midpoint of that gene and getting a distribution of these rates across the genome. But I am not sure if this is useful or conventional. Thoughts?


\textbf{Inversions and Gene Expression:}

I have been thinking about the inversions and gene expression project a lot this week and the only bacteria that has enough gene expression data is \ecol. So all analysis will have to be done on only that.
I have 3 strains of \ecol gene expression data: k-12, ATCC, and Saki. Within K-12, I have 3 substrains: MG1655, DH108 and BW25113. Which makes for a total of 5 genomes. 
I was thinking about maybe doing the same ancestral reconstruction process with these 5 strains but I am not sure of a few things:
\begin{itemize}
	\item Is 5 genomes going to provide enough data/information?
	\item would I use a midpoint for each gene and have one data point per node in the tree per gene?
	\item or should I have all positions in the gene have the same expression value?
\end{itemize}
Re-doing this analysis should not be too bad but would require me to re-code a few things to make the new data work. 

I was also wondering if an inversion and reverse complement of a sequence is the same thing? That is, does an inverted piece of sequence have to be the reverse complement to be re-inserted into the genome?
Another thing that I realized is that \p and PARSNP both allow for the blocks to be present in different parts of the genome for the different taxa. So this would be blending the rearrangements information and inversions information. Which then I am not sure if we can say that the gene expression differences are due to inversions alone unless we correct for this some how.
But overall, PARSNP and \p are identifying the same large inversions, the only difference is that PARSNP can identify very small inversions better.
I think this is because \p is trying to make the biggest blocks possible and is aligning the whole genome, where as PARSNP is just doing the core genome.
Another difference is that PARSNP has very very few gaps in its alignment, I think this a result from aligning just the core? Not sure how this will impact future results.


%For the papers solely on gene expression, they were all basically saying the things that we already know, that gene expression tends to decrease when moving away from the origin.
%However, there were varying explanations as to why this was happening. Some papers attributed this to an increase in gene dosage near the origin, but others said that this was not the case and there must be some other mechanism for controlling this or that gene expression trends could be a secondary effect of selection on say gene order or chromosome organization.
%But, they all basically said that because bacterial genomes are so highly organized based on gene order, physical folding of the chromosome, co-regulation of gene clusters...etc that this is why we see gene expression decreasing when moving away from the origin of replication.
%
%For the papers on Inversions, the results were a bit all over the place.
%Most of the papers were older (1980's) and often focused on one known inversion in one bacteria.
%These often were related to antibiotic resistance, flagella state of bacteria, or turning specific genes on/off.
%Sometimes these studies would say how the inversion altered expression of close down or upstream genes by changing promoters locations.
%Other studies engineered their own inversions, some mentioning gene expression and some don't and talk more about the replication of inversions or how they tend to be symmetrical around the origin.
%One large study done with Staphylococcus in 2012 just mentions that lots of genes are deferentially expressed (some up regulated and some down regulated).
%The overall feeling I got was that inversions can alter gene expression and this can be done by changing the promoter location of specific genes, which can impact many things about the bacteria like their growth state and resistance.
%There seems to be no ``trend'' with respect to inversions always causing gene expression to go up or down.
%It also looks like no one has done what we are doing: looking at existing expression data to see how having an inversion impacts gene expression within and outside of the inversion.

%I also looked at the inversions and gene expression for the 5 strains of \ecol and there are inversions (pics of alignments below, the taxa with all the weird stuff going on is ATCC)!!
%Mauve identified 3 large inversions roughly: 1,000,000bp, 800,000bp and 500,000bp. 
%Parsnp identified similar inversions (in similar locations) roughly: 1,350,000bp, 547,000bp and 1,376,300bp.
%Both of the programs had some sections that were not within an LCB or part of the core genome within these regions so the actual continuous length of the inversions may be smaller.
%These inversions only exist in one of the bacteria: \ecoli ATCC. The other 4 strains are all VERY similar with hardly any rearrangements, let alone inversions.
%So based on this we decided that we can just compaire the inversions between K-12 MG1655 and ATCC.
%I believe that \p is only showing information from one strand so when something is listed on the opposite strand from the reference in \p then it should be an inversion. I obviously need to check this and make sure this is the case. If it is, then it will make coding to identify inversions super simple!
%Also need to check if the inversions identified by \p and PARSNP are the same.

%%%%%%%%%%%%%%%%%bring these up next time!!!

%I realized that for \pb I miscalculated the bidirectionality transformation, so I had to fix this and re-run everything.
%It did not change the logistic regression results (seen below).
%However, when I re-did the origin shuffling to see if the placement of the origin changed anything, moving the origin 100kb, 90kb and 80kb to the left made the logistic regression negative (opposite). I have been trying to figure out why this is happening but I am having no luck. I thought maybe it was because these shufflings are now ~700kb away from the terminus, but the actual origin is about the same distance. I am still trying to figure this out but I am not sure what to do or what it means about the robustness of the origin shuffling.

%I looked at the gene expression data for \smel in detail and it appears to look ok.
%When I graph the raw data and plot the regression line it looks like there is no trend for \smel, the points are evenly distributed throughout the genome and there appears to be no increasing or decreasing trend.
%When looking at the other bacteria's raw data, there is clearly a decreasing trend when moving away from the origin.
%Additionally the number of genes and number of replicates are all comparable between \smel and the other bacteria.
%So I think that the reason \smel does not have significant gene expression regressions is because there is simply no trend.

%Last week I was reading an average of 2 articles a day (and will continue to do so until Aug 15). 
%While reading these papers I was researching the increased number of substitutions found near the origin of mitochondria.
%The papers I found on this subject are slightly misleading. The titles suggest that the substitution rate near the Control Region (CR) of mitochondria (which spans the origin on either side) is higher.
%However, after reading the articles I found that the substitution rate is higher than what was previously estimated using phylogenetic methods.
%So, I need to do a bit more research to see if I can find what the substitution rate for the rest of the mt genome is and see if the CR actually is higher.

\section*{This Week}
%aim to tick off 4 tasks a week
%
%This week I would like to figure out how to calculate dN/dS for the substitution data.
Continue to check in with the dN/dS stuff and make sure my per gene and genome rates are correct and try to interpret these!

I would like to obtain all inversions from Mauve or PARSNP alignment for the inversions and gene expression analysis.

I want to work on finishing up some scholarships for travel to SMBE.
 
\section*{Next Week}

Create histograms with the total number of genes in each 10kb section of the genome to supplement the gene expression analysis.

Continue working on the inversions and gene expression analysis.

Write up interpretation of dN/dS results.
\newpage


Box plots for per gene dN, dS, and $\omega$

\includegraphics[width=\textwidth]{C:/Users/Daniella/Documents/Sinorhizobium2015/Figs/dN_dS_box_plots_15May19/ecoli_dN_dS_omega_box_plot.pdf}
\includegraphics[width=\textwidth]{C:/Users/Daniella/Documents/Sinorhizobium2015/Figs/dN_dS_box_plots_15May19/bass_dN_dS_omega_box_plot.pdf}
\includegraphics[width=\textwidth]{C:/Users/Daniella/Documents/Sinorhizobium2015/Figs/dN_dS_box_plots_15May19/strep_dN_dS_omega_box_plot.pdf}
\includegraphics[width=\textwidth]{C:/Users/Daniella/Documents/Sinorhizobium2015/Figs/dN_dS_box_plots_15May19/sinoC_dN_dS_omega_box_plot.pdf}
\includegraphics[width=\textwidth]{C:/Users/Daniella/Documents/Sinorhizobium2015/Figs/dN_dS_box_plots_15May19/pSymA_dN_dS_omega_box_plot.pdf}
\includegraphics[width=\textwidth]{C:/Users/Daniella/Documents/Sinorhizobium2015/Figs/dN_dS_box_plots_15May19/pSymB_dN_dS_omega_box_plot.pdf}


%Below is a summary of what has been thrown out because of reasons (for one particular Block with an alignment length of 252,304 which includes gaps):
%\begin{table}[h]
%	\centering
%	\resizebox{0.8\textwidth}{!}{\begin{minipage}{\textwidth}
%			\begin{tabular}{lll}
%				\toprule
%				Category & Number of Genes& Proportion of Genes\\
%				\midrule
%				Total Genes  & 188 & 100\% \\
%				Genes not divisible by 3 & 50 & ~27\%\\
%				Genes with all gaps & 34 & ~18\% \\
%				Total Usable Genes & 104 & ~55\% \\
%				\bottomrule
%				
%				
%			\end{tabular}
%		\end{minipage}}
%	\end{table}	

\begin{table}[h]
	\centering
	\resizebox{\textwidth}{!}{%
		\begin{tabular}{lrrrrrr}
			\toprule
			& \multicolumn{3}{c}{Gene Average} & \multicolumn{3}{c}{Genome Average}\\
			\cmidrule{2-7}
%			\cmidrule{8-10}
%			& \multicolumn{3}{c}{Weighted} & \multicolumn{3}{c}{Non-weighted} & \multicolumn{3}{c}{Weighted} \\
%			\cmidrule{2-3}
%			\cmidrule{4-7}
			Bacteria and Replicon &  dS & dN & $\omega$ & dS & dN & $\omega$ \\
			\midrule
			\ecol Chromosome & 0.2924 & 0.0144& 0.0604 & 0.2600 & 0.0133 & 0.0556 \\
			\bass Chromosome & 0.6526& 0.0358& 0.0891& 0.5267 & 0.0321& 0.0828\\
			\strep Chromosome &  0.1924 & 0.3201& 2.6404& 0.1775 & 0.3017 & 2.4358 \\
			\smel Chromosome &  0.0134  & 0.0014& 0.0844& 0.0134& 0.0013& 0.0930 \\
			\smel pSymA & 0.0798 & 0.0109& 0.2320& 0.0800 & 0.0103 & 0.2218\\
			\smel pSymB & 0.0814 & 0.0086& 0.1639& 0.0782 & 0.0082 & 0.1590\\
			\bottomrule
		\end{tabular}
		
	}%resizebox
	\caption{\label{tab:dN_dS_ratios} Weighted averages calculated for each bacterial replicon on a per genome basis using the gene length as the weight. Arithmetic mean calculated for the per gene averages for each bacterial replicon.}
\end{table}

%\begin{table}[h]
%	\centering
%	\resizebox{\textwidth}{!}{%
%		\begin{tabular}{lrrrrrrrrr}
%			\toprule
%			& \multicolumn{6}{c}{Gene Average} & \multicolumn{3}{c}{Genome Average}\\
%			\cmidrule{2-7}
%			\cmidrule{8-10}
%			& \multicolumn{3}{c}{Weighted} & \multicolumn{3}{c}{Non-weighted} & \multicolumn{3}{c}{Weighted} \\
%			\cmidrule{2-3}
%			\cmidrule{4-7}
%			Bacteria and Replicon & dS & dN & $\omega$  & dS & dN & $\omega$ & dS & dN & $\omega$ \\
%			\midrule
%			\ecol Chromosome & 0.2600 & 0.0133 & 0.0556 & 0.2924 & 0.0144& 0.0604 & 0.2600 & 0.0133 & 0.0556 \\
%			\bass Chromosome & 0.5267& 0.0321& 0.0828& 0.6526& 0.0358& 0.0891& 0.5267 & 0.0321& 0.0828\\
%			\strep Chromosome & 0.1775& 0.3017& 2.4358& 0.1924 & 0.3201& 2.6404& 0.1775 & 0.3017 & 2.4358 \\
%			\smel Chromosome & 0.0134& 0.0013& 0.0930& 0.0134  & 0.0014& 0.0844& 0.0134& 0.0013& 0.0930 \\
%			\smel pSymA & 0.0800& 0.0103& 0.2218& 0.0798 & 0.0109& 0.2320& 0.0800 & 0.0103 & 0.2218\\
%			\smel pSymB & 0.0782& 0.0082& 0.1590& 0.0814 & 0.0086& 0.1639& 0.0782 & 0.0082 & 0.1590\\
%			\bottomrule
%		\end{tabular}
%		
%	}%resizebox
%	\caption{\label{tab:dN_dS_ratios} Weighted averages calculated for each bacteria on a per gene and per genome basis. Averages are weighted by gene length for the per gene averages, and by section length for the per genome averages.}
%\end{table}

\begin{table}[h]
	\centering
	\resizebox{\textwidth}{!}{%
		\begin{tabular}{lrrrrr}
			\toprule
			Bacteria and Replicon & Average Replicon Length & \# of Coding Sites & \# of Non-Coding Sites & \# of Subs Coding & \# of Subs Non-Coding\\
			\midrule
			\ecol Chromosome & 5082529 & 2960007 & 191748 & 207199 & 9534 \\ % 4048616/4641652 & 
			\bass Chromosome & 4077077 & 2074653 & 102906 & 205150 & 6187\\ %3692179/4215606
			\strep Chromosome & 8497577 & 2422980 & 21581 & 551530 & 3670\\ % 7628849/8667507
			\smel Chromosome & 3426881 & 1931139 & 199425 & 6684 & 842\\ % 3130925/3654135
			\smel pSymA & 1455940 & 419223 & 34213 & 9832 & 943\\ % 1128615/1354226 & 225611/1354226
			\smel pSymB & 1664597 & 552816 & 22098 & 11699 & 645\\ % 1493048/1683333
			\bottomrule
		\end{tabular}
		
	}%resizebox
	\caption{\label{tab:cod_non_cod_proportions} Total proportion of coding and non-coding sites in each replicon and the percentage of those sites that have a substitution (multiple substitutions at one site are counted as two substitutions).}
\end{table}



\begin{table}[h]
	\centering
	\resizebox{\textwidth}{!}{%
		\begin{tabular}{lcc}
			\toprule
			Bacteria and Replicon & Coding Sequences & Non-Coding Sequences\\
			\midrule
			\ecol Chromosome & \cc -9.983\e{-8}*** &  6.994\e{-8}*** \\
			\bass Chromosome & \cc-1.071\e{-7}***  & \cc -9.861\e{8}*** \\
			\strep Chromosome & \cc -2.626\e{-8}***  & 3.615\e{-7}*** \\
			\smel Chromosome & \cc -1.367\e{-7}***  & \cc -1.510\e{-7}* \\
			\smel pSymA & \cc -1.075\e{-7}* & NS \\
			\smel pSymB & 2.878\e{-7}*** & 8.595\e{-7}***\\
			\bottomrule
		\end{tabular}
		
	}%resizebox
	\caption{\label{tab:cod_non_cod_log_reg} Logistic regression analysis of the number of substitutions along all positions of the genome of the respective bacteria replicons. These genomic positions were split up into the coding and non-coding regions of the genome. Grey coloured boxes indicate a negative logistic regression coefficient estimate. All results are statistically significant. Logistic regression was calculated after the origin of replication was moved to the beginning of the genome and all subsequent positions were scaled around the origin accounting for bidirectionality of replication. All results are marked with significance codes as followed: $<$ 0.001 = `***', 0.001 $<$ 0.01 = `**', 0.01 $<$ 0.05 = `*', $>$ 0.05 = `NS'.}
\end{table}
	

Sub density graphs with coding and non-coding information

\includegraphics[width=\textwidth]{C:/Users/Daniella/Documents/Sinorhizobium2015/Figs/Sub_graphs_cod_non_cod_7Feb19/ecoli_chrom_cod_non_cod_sub_histogram_bidirectionality_colour_9May19.pdf}

\includegraphics[width=\textwidth]{C:/Users/Daniella/Documents/Sinorhizobium2015/Figs/Sub_graphs_cod_non_cod_7Feb19/bass_chrom_cod_non_cod_sub_histogram_bidirectionality_colour_7May19.pdf}

\includegraphics[width=\textwidth]{C:/Users/Daniella/Documents/Sinorhizobium2015/Figs/Sub_graphs_cod_non_cod_7Feb19/strep_chrom_cod_non_cod_sub_histogram_bidirectionality_colour_8May19.pdf}

\includegraphics[width=\textwidth]{C:/Users/Daniella/Documents/Sinorhizobium2015/Figs/Sub_graphs_cod_non_cod_7Feb19/sinoC_chrom_cod_non_cod_sub_histogram_bidirectionality_colour_7May19.pdf}

\includegraphics[width=\textwidth]{C:/Users/Daniella/Documents/Sinorhizobium2015/Figs/Sub_graphs_cod_non_cod_7Feb19/pSymA_chrom_cod_non_cod_sub_histogram_bidirectionality_colour_7May19.pdf}

\includegraphics[width=\textwidth]{C:/Users/Daniella/Documents/Sinorhizobium2015/Figs/Sub_graphs_cod_non_cod_7Feb19/pSymB_chrom_cod_non_cod_sub_histogram_bidirectionality_colour_7May19.pdf}

Gene expression graphs

% Gene Exp graphs
\includegraphics[width=\textwidth]{C:/Users/Daniella/Documents/Sinorhizobium2015/Gene_Expression_8Dec17/Gene_Expression/Graphs/ecoli_chrom_sub_exp_histogram_bidirectionality_colour_13Mar19.pdf}

\includegraphics[width=\textwidth]{C:/Users/Daniella/Documents/Sinorhizobium2015/Gene_Expression_8Dec17/Gene_Expression/Graphs/bass_chrom_sub_exp_histogram_bidirectionality_colour_4Mar19.pdf}

\includegraphics[width=\textwidth]{C:/Users/Daniella/Documents/Sinorhizobium2015/Gene_Expression_8Dec17/Gene_Expression/Graphs/strep_chrom_sub_exp_histogram_bidirectionality_colour_4Mar19.pdf}

\includegraphics[width=\textwidth]{C:/Users/Daniella/Documents/Sinorhizobium2015/Gene_Expression_8Dec17/Gene_Expression/Graphs/smel_chrom_sub_exp_histogram_bidirectionality_colour_4Mar19.pdf}

\includegraphics[width=\textwidth]{C:/Users/Daniella/Documents/Sinorhizobium2015/Gene_Expression_8Dec17/Gene_Expression/Graphs/pSymA_sub_exp_histogram_bidirectionality_colour_4Mar19.pdf}

\includegraphics[width=\textwidth]{C:/Users/Daniella/Documents/Sinorhizobium2015/Gene_Expression_8Dec17/Gene_Expression/Graphs/pSymB_sub_exp_histogram_bidirectionality_colour_4Mar19.pdf}




\includegraphics[width=\textwidth]{Mauve_aln_pic_17Dec18.jpg}

\includegraphics[width=\textwidth]{Parsnp_aln_pic_17Dec18.png}


%\begin{table}[h]
%	\centering
%	\resizebox{\textwidth}{!}{%
%		\begin{tabular}{lccrr}
%			\toprule
%			Bacteria and Replicon & \% of Coding Sequences & \% of Non-Coding Sequences & \% of Subs Coding & \% of Subs Non-Coding\\
%			\midrule
%			\ecol Chromosome & 86.47\% & 13.53\% & 5.00\% & 8.96\% \\ % 4013445/4641652 & 200860 & 56265 
%			\bass Chromosome & 87.49\% & 12.51\% & 7.31\% & 6.42\%\\ %3688210/4215606 & 269482 & 33846
%			\strep Chromosome & 89.03\% & 10.97\% & 13.74\% & 14.91\%\\ % 7716382/8667507 & 1060278 & 141828
%			\smel Chromosome & 86.27\% & 13.73\% & 0.19\% & 0.22\%\\ % 3152315/3654135 & 6023 & 1088
%			\smel pSymA & 83.34\% & 16.66\% & 2.84\% & 4.58\%\\ % 1128615/1354226 & 225611/1354226 & 32040 & 10333
%			\smel pSymB & 88.81\% & 11.19\% & 2.78\% & 3.44\%\\ % 1495047/1683333 & 41532 & 6483
%			\bottomrule
%		\end{tabular}
%		
%	}%resizebox
%	\caption{\label{tab:cod_non_cod_proportions} Total proportion of coding and non-coding sites in each replicon and the percentage of those sites that have a substitution (multiple substitutions at one site are counted as two substitutions).}
%\end{table}

%
%

%
%
%\begin{table}[h]
%	\centering
%	\resizebox{\textwidth}{!}{%
%		\begin{tabular}{lccc}
%			\toprule
%			Bacteria and Replicon & Coefficient Estimate & Standard Error & P-value \\
%			\midrule
%			\ecol Chromosome & 2.496\e{-5} & 8.695\e{-6}& 0.0041 \\
%			\bass Chromosome & 1.912\e{-6} & 8.753
%			\e{-8} & $<$2\e{-16}\\
%			\strep Chromosome & 2.984\e{-5} & 1.858\e{-6} & $<$2\e{-16}\\
%			\smel Chromosome & 6.993\e{-6} & 6.205\e{-7} & $<$2\e{-16}\\
%			\smel pSymA & -9.713\e{-7} & 3.212\e{-8} & $<$2\e{-16}\\
%			\smel pSymB & 6.181\e{-7} & 2.253\e{-8} & $<$2\e{-16}\\
%			\bottomrule
%		\end{tabular}
%		
%	}%resizebox
%	\caption{\label{tab:cod_log_reg} Logistic regression analysis of the number of substitutions along all coding portions of the genome of the respective bacteria replicons. Grey coloured boxes indicate a negative logistic regression coefficient estimate. All results are statistically significant. Logistic regression was calculated after the origin of replication was moved to the beginning of the genome and all subsequent positions were scaled around the origin accounting for bidirectionality of replication.}
%\end{table}
%
%\begin{table}[h]
%	\centering
%	\resizebox{\textwidth}{!}{%
%		\begin{tabular}{lccc}
%			\toprule
%			Bacteria and Replicon & Coefficient Estimate & Standard Error & P-value \\
%			\midrule
%			\ecol Chromosome & -1.397\e{-7} & 2.427\e{-9} & $<$ 2\e{-16} \\
%			\bass Chromosome & -1.439\e{-8} & 1.569\e{-9} & $<$2\e{-16}\\
%			\strep Chromosome & 1.689\e{-8} & 7.235\e{-10} & $<$2\e{-16}\\
%			\smel Chromosome & -1.311\e{-6}& 3.393\e{-8}& $<$2\e{-16}\\
%			\smel pSymA & -1.413\e{-7} & 3.762\e{-8} & 1.73\e{-4}\\
%			\smel pSymB & 5.196\e{-7} & 4.769\e{-8} & $<$2\e{-16}\\
%			\bottomrule
%		\end{tabular}
%		
%	}%resizebox
%	\caption{\label{tab:non_cod_log_reg} Logistic regression analysis of the number of substitutions along all non-coding portions of the genome of the respective bacteria replicons. Grey coloured boxes indicate a negative logistic regression coefficient estimate. All results are statistically significant. Logistic regression was calculated after the origin of replication was moved to the beginning of the genome and all subsequent positions were scaled around the origin accounting for bidirectionality of replication.}
%\end{table}

\begin{table}[h]
	\centering
	\resizebox{0.8\textwidth}{!}{\begin{minipage}{\textwidth}
			\begin{tabular}{lll}
				\toprule
				Bacteria Strain/Species & GEO Accession Number & Date Accessed\\
				\midrule
				\ecol K12 MG1655  & GSE60522 & December 20, 2017\\
				\ecol K12 MG1655 & GSE73673 & December 19, 2017\\
				\ecol K12 MG1655 & GSE85914 & December 19, 2017\\
				\ecol K12 MG1655 & GSE40313 & November 21, 2018\\
				\ecol K12 MG1655 & GSE114917 & November 22, 2018\\
				\ecol K12 MG1655 & GSE54199 & November 26, 2018\\
				\ecol K12 DH10B & GSE98890 & December 19, 2017\\
				\ecol BW25113 & GSE73673 & December 19, 2017 \\
				\ecol BW25113 & GSE85914 & December 19, 2017 \\
				\ecol O157:H7 & GSE46120 & August 28, 2018\\
				\ecol ATCC 25922 & GSE94978 & November 23, 2018\\
% ecoli has lots of aerobic anerobic...etc growth phases, and time periods
				\midrule
				\bass 168 & GSE104816 & December 14, 2017\\
				\bass 168 & GSE67058 & December 16, 2017\\
				\bass 168 & GSE93894 & December 15, 2017\\
				\bass 168 & GSE80786 & November 16, 2018\\
				% bass also has quite a few time dependent data sets, or separated by growth phase				
				\midrule
				\scoe A3 & GSE57268 & March 16, 2018\\
				\snat HW-2 & GSE112559 & November 15, 2018\\
				% strep has LOTS of time dependent data sets, but the other bac do not really have that				
				\midrule
				\smel 1021 Chromosome & GSE69880 & December 12, 2017\\
				%				\smel 1021 Chromosome & GSE121228 & diff phases\\
				\midrule
				\smel 2011 \pa & NC\_020527 (Dr. Finan) & April 4, 2018 \\
				\smel 1021 \pa & GSE69880 & November 15, 18\\
				\midrule
				\smel 2011 \pb & NC\_020560 (Dr. Finan) & April 4, 2018 \\
				\smel 1021 \pb & GSE69880 & November 15, 18\\
				\bottomrule
				
				
			\end{tabular}
			\caption{\label{tab:tableS1} Summary of strains and species found for each gene expression analysis. Gene Expression Omnibus accession numbers and date accessed are provided.}
		\end{minipage}}
	\end{table}	





%\begin{table}[]
%	\centering
%	\resizebox{\textwidth}{!}{%
%		\begin{tabular}{lcccccc}
%			\toprule
%			Position Difference & \ecol Chromosome &\bass Chromosome & \strep Chromosome & \smel Chromosome &\smel pSymA & \smel pSymB\\
%	\midrule
%	1bp & -1.394\e{-7}** & -2.538\e{-8}** & 1.736\e{-8}** & -1.541\e{-6}** & -9.130\e{-7}** & 2.488\e{-7}*** \\
%	10bp & -1.394\e{-7}*** & -2.518\e{-8}*** & -4.484\e{-9}*** & -1.627\e{-6}*** & -9.13\e{-7}*** & 3.487\e{-7}***\\
%	100bp & -1.764\e{-7}*** & -1.417\e{-8}*** & 1.448\e{-8}*** & -1.605\e{-6}*** & -1.166\e{-6}*** & 4.021\e{-7}*** \\
%	1000bp & -1.784\e{-7}*** & -1.417\e{-8}*** & 1.505\e{-8}*** & -1.605\e{-6}*** & -1.153\e{-6}*** & 4.021\e{-7}***\\
%	10000bp & -1.712\e{-7}*** & -3.496\e{-8}*** & 4.790\e{-8}*** & -1.605\e{-6}*** & -3.570\e{-8}* & 3.784\e{-7}*** \\
%	100000bp & -2.061\e{-7}*** & -3.561\e{-8}*** & 4.167\e{-9}*** & -1.605\e{-6}*** & -4.676\e{-7}*** & 3.784\e{-7}***\\
%	1000000bp & 4.229\e{-8}*** & -7.710\e{-9}*** & 6.083\e{-8}*** &-1.605\e{-6}*** & 4.285\e{-6}*** & -8.888\e{-7}*** \\
%	\bottomrule
%		\end{tabular}
%		
%	}%resizebox
%	\caption{\label{tab:clustering} Position clustering analysis. Logistic regression analysis of the number of substitutions along the genome of the respective bacteria replicons to test position differences. Each row denotes different base pair distances that the positions were clustered together as. All results are marked with significance codes as followed: $<$ 0.001 = `***', 0.001 $<$ 0.01 = `**', 0.01 $<$ 0.05 = `*', 0.05 $<$ 0.1 = `.', $>$ 0.1 = ` '. Logistic regression was calculated after the positions in the genome were determined to be the same at each position difference listed in the first column.}
%\end{table}
%
%
\begin{table}[h]
	\centering
	\resizebox{\textwidth}{!}{%
		\begin{tabular}{lccc}
			\toprule
			Bacteria and Replicon & Coefficient Estimate & Standard Error & P-value \\
			\midrule
			\cellcolor{black!16}\ecol Chromosome & \cellcolor{black!16}-6.03\e{-5} & \cellcolor{black!16}1.28\e{-5} & \cellcolor{black!16}2.8\e{-6} \\
			\cellcolor{black!16}\bass Chromosome & \cellcolor{black!16}-9.7\e{-5} & \cellcolor{black!16}2.0\e{-5} & \cellcolor{black!16}1.2\e{-6} \\
			\cellcolor{black!16}\strep Chromosome & \cellcolor{black!16}-1.17\e{-6} & \cellcolor{black!16}1.04\e{-7} & \cellcolor{black!16}$<$2\e{-16}\\
			\smel Chromosome & 3.97\e{-5} & 4.25\e{-5} & NS (3.5\e{-1})\\
			\smel pSymA & 1.39\e{-3} & 2.53\e{-4} & 4.9\e{-8} \\
			\smel pSymB & 1.46\e{-4} & 2.03\e{-4} & NS (5.34.7\e{-1})\\
			\bottomrule
		\end{tabular}
		
	}%resizebox
	\caption{\label{tab:lr_exp} Linear regression analysis of the median counts per million expression data along the genome of the respective bacteria replicons. Grey coloured boxes indicate statistically significant results at the 0.5 significance level. Linear regression was calculated after the origin of replication was moved to the beginning of the genome and all subsequent positions were scaled around the origin accounting for bidirectionality of replication.}
\end{table}

%\begin{table}[b!]
%	\centering
%	\resizebox{\textwidth}{!}{%
%		\begin{tabular}{lccc}
%			\toprule
%			Bacteria and Replicon & Coefficient Estimate & Standard Error & P-value \\
%			\midrule
%			\cellcolor{black!16}\ecol Chromosome & \cellcolor{black!16}-1.394\e{-7} & \cellcolor{black!16}2.425\e{-9} & \cellcolor{black!16}$<$2\e{-16} \\
%			\cellcolor{black!16}\bass Chromosome & \cellcolor{black!16}-1.265\e{-8} & \cellcolor{black!16}1.562\e{-9} & \cellcolor{black!16}5.430\e{-16} \\
%			\strep Chromosome & 1.736\e{-8} & 7.231\e{-10} & $<$2\e{-16}\\
%			\cellcolor{black!16}\smel Chromosome & \cellcolor{black!16}-1.541\e{-6} & \cellcolor{black!16}3.042\e{-8} & \cellcolor{black!16}$<$2\e{-16}\\
%			\cellcolor{black!16}\smel pSymA & \cellcolor{black!16}-9.130\e{-7} & \cellcolor{black!16}1.975\e{-8} & \cellcolor{black!16}$<$2\e{-16} \\
%			\smel pSymB & 2.488\e{-7} & 1.964\e{-8} & $<$2\e{-16}\\
%			\bottomrule
%		\end{tabular}
%		
%	}%resizebox
%	\caption{\label{tab:tabel2} Logistic regression analysis of the number of substitutions along the genome of the respective bacteria replicons. Grey coloured boxes indicate a negative logistic regression coefficient estimate. All results are statistically significant. Logistic regression was calculated after the origin of replication was moved to the beginning of the genome and all subsequent positions were scaled around the origin accounting for bidirectionality of replication.}
%\end{table}
%
%
%
%\includegraphics[width=\textwidth]{C:/Users/Daniella/Documents/Sinorhizobium2015/Gene_Expression_8Dec17/Gene_Expression/smel_chrom_sub_exp_histogram_bidirectionality_colour_16Mar18}
%
%\includegraphics[width=\textwidth]{C:/Users/Daniella/Documents/Sinorhizobium2015/Gene_Expression_8Dec17/Gene_Expression/pSymA_sub_exp_histogram_bidirectionality_colour_9Apr18}
%
%\includegraphics[width=\textwidth]{C:/Users/Daniella/Documents/Sinorhizobium2015/Gene_Expression_8Dec17/Gene_Expression/pSymB_sub_exp_histogram_bidirectionality_colour_12Apr18}
%
%\includegraphics[width=\textwidth]{C:/Users/Daniella/Documents/Sinorhizobium2015/Gene_Expression_8Dec17/Gene_Expression/bass_chrom_sub_exp_histogram_bidirectionality_colour_5Apr18}
%
%\includegraphics[width=\textwidth]{C:/Users/Daniella/Documents/Sinorhizobium2015/Gene_Expression_8Dec17/Gene_Expression/ecoli_chrom_sub_exp_histogram_bidirectionality_colour_16Mar18}
%
%\includegraphics[width=\textwidth]{C:/Users/Daniella/Documents/Sinorhizobium2015/Gene_Expression_8Dec17/Gene_Expression/strep_chrom_sub_exp_histogram_bidirectionality_colour_19Mar18}
%
%%\begin{table}[h]
%%	\centering
%%	\resizebox{\textwidth}{!}{%
%%		\begin{tabular}{lccc}
%%			\toprule
%%			Bacteria and Replicon & Coefficient Estimate & Standard Error & P-value \\
%%			\midrule
%%			\ecol Chromosome & -1.44\e{-7}& 2.01\e{-9} & $<$2\e{-16}\\
%%			\bass Chromosome & -1.121\e{-7} & 3.41\e{-9} & $<$2\e{-16}\\
%%			\strep Chromosome & 1.24\e{-8} & 7.2\e{-10} & $<$2\e{-16}\\
%%			\smel Chromosome & -1.526\e{-6} & 3.02\e{-8} & $<$2\e{-16}\\
%%			\smel pSymA & -1.058\e{-6}& 2.58\e{-8}& $<$2\e{-16}\\
%%			\smel pSymB & 1.79\e{-7}& 1.84\e{-8}& 1.6\e{-10}\\
%%			\bottomrule
%%		\end{tabular}
%%		
%%	}%resizebox
%%	\caption{\label{tab:tabel2} Logistic regression analysis of the number of substitutions along the genome of the respective bacteria replicons when 10,000bp sections of the genome are re-organized.}
%%\end{table}
%
%
%\begin{table}[]
%	\centering
%	\resizebox{1.2\textwidth}{!}{%
%			\begin{tabular}{lcccccc}
%\toprule
%Origin Location & \ecol Chromosome &\bass Chromosome & \strep Chromosome & \smel Chromosome &\smel pSymA & \smel pSymB\\
%\midrule
%Moved 100kb Left & -1.445\e{-7}*** & 4.374\e{-9}* &  6.909\e{-9}*** &  -1.316\e{-6}*** & -1.058\e{-6}*** & -2.009\e{-7}***\\
%Moved 90kb Left & -1.544\e{-7}*** & -1.036\e{-7}*** & 5.677\e{-9}*** & -1.32\e{-6}*** & -1.246\e{-6}*** &-1.357\e{-7}***\\
%Moved 80kb Left& -1.65\e{-7}*** & -1.072\e{-7}*** & 8.11\e{-9}*** & -1.338\e{-6}*** & -1.398\e{-6}*** & -6.57\e{-8}***\\
%Moved 70kb Left& -1.667\e{-7}*** & -1.102\e{-7}*** & 6.716\e{-9}*** & -1.363\e{-6}*** & -1.405\e{-6}*** & 9.83\e{-8}\\
%Moved 60kb Left& -1.64\e{-7}*** & -1.19\e{-7}*** & 8.7\e{-9}*** & -1.324\e{-6}*** & -1.394\e{-6}*** & 1.129\e{-7}***\\
%Moved 50kb Left& -1.446\e{-7}*** & -1.211\e{-7}*** & 1.045\e{-8}*** & -1.36\e{-6}*** & -1.403\e{-6}*** & 1.521\e{-7}***\\
%Moved 40kb Left& -1.4\e{-7}*** & -1.299\e{-7}*** & 1.214\e{-8}*** & -1.255\e{-6}*** & -1.422\e{-6}*** & 1.543\e{-7}***\\
%Moved 30kb Left& -1.498\e{-7}*** & -1.292\e{-7}*** & 1.24\e{-8}*** & -1.26\e{-6}*** & -1.392\e{-6}*** & 1.63\e{-7}***\\
%Moved 20kb Left& -1.51\e{-7}*** & -1.1\e{-7}*** & 1.395\e{-8}*** & -1.525\e{-6}*** & -1.412\e{-6}*** & 1.603\e{-7}***\\
%Moved 10kb Left& -1.262\e{-7}*** & -2.602\e{-9} & 1.563\e{-8}*** &  -1.599\e{-6}*** &  -9.499\e{-7}*** & 2.973\e{-7}*** \\
%Moved 10kb Right& -1.305\e{-7}*** & -2.045\e{-8}*** & 1.578\e{-8}*** & 1.614\e{-6}*** & -1.026\e{-6}*** & 3.505\e{-7}*** \\
%Moved 20kb Right& -1.454\e{-7}*** & -1.006\e{-7}*** & 1.903\e{-8}*** & -1.634\e{-6}*** & -1.475\e{-6}*** & 1.649\e{-7}***\\
%Moved 30kb Right& -1.548\e{-7}*** & -8.596\e{-8}*** & 2.046\e{-8}*** & -1.698\e{-6}*** & -1.417\e{-6}*** & 1.526\e{-7}***\\
%Moved 40kb Right& -1.632\e{-7}*** & -8.378\e{-8}*** & 2.125\e{-8}*** & -1.719\e{-6}*** & -1.367\e{-6}*** & 1.589\e{-7}***\\
%Moved 50kb Right& -1.856\e{-7}*** & -7.879\e{-8}*** & 1.957\e{-8}*** & -1.735\e{-6}*** & -1.277\e{-6}*** & 1.654\e{-7}***\\
%Moved 60kb Right& -1.91\e{-7}*** & -6.98\e{-8}*** & 1.974\e{-8}*** & -1.788\e{-6}*** & -1.169\e{-6}*** & 1.645\e{-7}***\\
%Moved 70kb Right& -1.892\e{-7}*** & -6.634\e{-8}*** & 1.934\e{-8}*** & -1.854\e{-6}*** & -1.059\e{-6}*** & 1.843\e{-7}***\\
%Moved 80kb Right& -1.879\e{-7}** & -5.814\e{-8}*** & 2.313\e{-8}*** & -1.891\e{-6}*** & -9.07\e{-7}*** & 1.90\e{-7}***\\
%Moved 90kb Right& -1.862\e{-7}*** & -4.314\e{-8}*** & 2.304\e{-8}*** & -1.865\e{-6}*** & -7.171\e{-7}*** & 2.415\e{-7}***\\
%Moved 100kb Right& -1.799\e{-7}*** & -2.597\e{-8}*** &  1.945\e{-8}*** &  -1.525\e{-6}*** & -6.572\e{-7}*** & 3.095\e{-7}***\\
%\bottomrule
%\end{tabular}
%		
%	}%resizebox
%	\caption{\label{tab:tabel2} Logistic regression analysis of the number of substitutions along the genome of the respective bacteria replicons. All results are marked with significance codes as followed: $<$ 0.001 = `***', 0.001 $<$ 0.01 = `**', 0.01 $<$ 0.05 = `*', 0.05 $<$ 0.1 = `.', $>$ 0.1 = ` '. Logistic regression was calculated after the origin of replication was moved to the beginning of the genome and all subsequent positions were scaled around the origin accounting for bidirectionality of replication.}
%\end{table}
%
%%\begin{table}[h]
%%	\resizebox{\textwidth}{!}{%
%%		\begin{tabular}{lccc}
%%			\toprule
%%			Bacteria Replicon & \multicolumn{1}{p{3cm}}{\centering \% of Total LCBs \\ with Identical Tree} & 
%%			\multicolumn{1}{p{3cm}}{\centering \% of Total LCBs \\ with Not Identical Tree} & 
%%			\multicolumn{1}{p{3cm}}{\centering \% of Total Alignment Discarded} \\  
%%			\midrule
%%			\ecol Chromosome & 81.58\% & 18.42\% & 25.18\%\\
%%			\bass Chromosome & 83.33\% & 16.67\% & 19.37\%\\
%%			\strep Chromosome & 96.53\% & 3.47\% & 12.42\%\\
%%			\smel Chromosome & 81.82\% & 18.18\% & 25.42\%\\
%%			\smel \pa & 100\% & 0\% & 0\%\\
%%			\smel \pb & 100\% & 0\% & 0\%\\
%%			\bottomrule
%%		\end{tabular}
%%	}%resize box
%%\end{table}
%%
%% 
%
%%
%%\clearpage
%%
%%\includegraphics[width=\textwidth]{C:/Users/Daniella/Documents/Sinorhizobium2015/Figs/Bidirectionality_outliers_coloured_9Jun17/bass_chrom_change_histogram_bidirectionality_colour_6Nov17.pdf}
%%
%%\includegraphics[width=\textwidth]{C:/Users/Daniella/Documents/Sinorhizobium2015/Figs/Bidirectionality_outliers_coloured_9Jun17/streo_chrom_change_histogram_bidirectionality_colour_6Nov17.pdf}
%%
%%\includegraphics[width=\textwidth]{C:/Users/Daniella/Documents/Sinorhizobium2015/Figs/Bidirectionality_outliers_coloured_9Jun17/chrom_change_histogram_bidirectionality_colour_24Nov17.pdf}
%%
%%\includegraphics[width=\textwidth]{C:/Users/Daniella/Documents/Sinorhizobium2015/Papers/Substitutions_paper/Substitutions_paper/Figs/pSymA_change_histogram_bidirectionality_colour_6Oct17.pdf}
%%
%%\includegraphics[width=\textwidth]{C:/Users/Daniella/Documents/Sinorhizobium2015/Papers/Substitutions_paper/Substitutions_paper/Figs/pSymB_change_histogram_bidirectionality_colour_10Oct17.pdf}
%%
%%
\end{document}
