\documentclass[12pt]{article}
\usepackage{scrtime} % for \thistime (this package MUST be listed first!)
\usepackage[margin=0.75in]{geometry}
\usepackage{graphicx}
\usepackage{fancyhdr}
\usepackage{xspace}
%\usepackage{underscore}
\usepackage{pdfpages}
\usepackage{xcolor,colortbl}%for changing cell colour
\usepackage{longtable}
\usepackage{booktabs}
\pagestyle{fancy}
%\setlength{\headheight}{15.2pt}
\setlength{\headsep}{13 pt}
\setlength{\parindent}{28 pt}
\setlength{\parskip}{12 pt}
\pagestyle{fancyplain}
\usepackage[T1]{fontenc}
\usepackage{tikz-cd}
\usepackage{tikz}
\usepackage[normalem]{ulem} %to strikeout text
\usetikzlibrary{decorations.markings}
\usetikzlibrary{calc, arrows}
\usepackage{lscape} %to make the page landscape
\usepackage{color,amsmath,amssymb,amsthm,mathrsfs,amsfonts,dsfont}
\usepackage{indentfirst} % to indent the first paragraph
\rhead{\fancyplain{}{Thesis Update \today \hfill Daniella Lato}}
\title{Sinorhizobium Update}
\author{Daniella Lato}
\date{\today}
\renewcommand\headrulewidth{0.5mm}
\newcommand{\cc}{\cellcolor{black!16}}
\newcommand{\s}{\textit{Sinorhizobium}\xspace}
\newcommand{\smel}{\textit{S.\,meliloti}\xspace}
\newcommand{\smed}{\textit{S.\,medicae}\xspace}
\newcommand{\sfred}{\textit{S.\,fredii}\xspace}
\newcommand{\ssah}{\textit{S.\,saheli}\xspace}
\newcommand{\ster}{\textit{S.\,terangae}\xspace}
\newcommand{\agro}{\textit{A.\,tumefaciens}\xspace}
\newcommand{\escoli}{\textit{Escherichia coli}\xspace}
\newcommand{\bur}{\textit{Burkholderia}\xspace}
\newcommand{\vib}{\textit{Vibrio}\xspace}
\newcommand{\sul}{\textit{Sulfolobus}\xspace}
\newcommand{\ent}{\textit{Enterobacteria}\xspace}
\newcommand{\p}{progressiveMauve}
\newcommand{\bas}{\textit{Bacillus subtilis}\xspace}
\newcommand{\strep}{\textit{Streptomyces}\xspace}
\newcommand{\bass}{\textit{B.\,subtilis}\xspace}
\newcommand{\ecol}{\textit{E.\,coli}\xspace}
\newcommand{\ecoli}{\textit{Escherichia coli}\xspace}
\newcommand{\tub}{\textit{Mycobacterium tuberculosis}\xspace}
\newcommand{\pa}{pSymA\xspace}
\newcommand{\pb}{pSymB\xspace}
\newcommand{\snat}{\textit{S.\,natalensis}\xspace}
\newcommand{\scoe}{\textit{S.\,coelicolor}\xspace}
\providecommand{\e}[1]{\ensuremath{\times 10^{#1}}}

\begin{document}
%	Nov 30:	Create graphs with slopes for each COG
	
	
%	Dec 3:	Create new binned scatter plot of COG log reg
	
%	Dec 6:	Determine if there are any other stats I want/need for COG stuff
	
%	Dec 10:	Calculate above stats and write in table
	
%	Dec 21:	Find papers on COG stuff (for intro and discussion), and other mol trends (discussion for sub paper)
	
%	Jan 6:	Read above mentioned papers and make notes
	
%$\checkmark$	Aug 21: Comprehensive Exam 10:30am
%
%$\checkmark$ Aug 26: make new list of dates for goals
%
%$\checkmark$	Sep 7: Write Up methods for clustering testing and add to substitutions paper
%	
%$\checkmark$	Sep 14: Have Lab Meeting Presentation done
%	
%$\checkmark$	Sep 18: Present in Lab Meeting
%
%$\checkmark$	Sep 30: Have all clustering testing complete for all bacteria
%
%$\checkmark$	Sep 30: Compile notes from comps papers into one document
%		
%$\checkmark$	Oct 5: Gene expression data for the inversions project
%	
%	Oct 12: New intro for Substitution paper
%	
%$\checkmark$	Oct 5-12: Apply for Mac Scholarships and Awards
%	
%$\checkmark$ 	Oct 26: Write detailed outlines for all papers
%
%	Oct 31: Write out methods for gene expression paper
%	
%	Sep 9: Think about/compile list of inversions in \ecol for new paper
%	
%	Nov 15: Think about how to better look at the COG data
%	
%	Nov 25: Complete any extra analysis needed for Substitution paper
%	
%	Dec 4: Mac Scholarships and Awards Due
%	
%	Dec 1: Write out COG methods
%	
%	Dec 15: Gather papers for COG paper intro
%	
%	Dec 15: Implement COG stuff
%	
%	Other things to do:
%	
%	Create outline for gene expression paper
%
%  research mito increased subs near origin
%
%  add above to subs paper writeup
%	
%	Write Write Write gene expression paper
%	
%	 Have gene expression/inversion data combined and in graphical format/regression lines calculated
%	
%	re-do gene exp/sub graphs with patchwork R package so they line up exactly
%	
%	Have data for other molecular trends (GC content, number of genes, essential gene lists..etc.) combined with graphs (or in supplement) for sub analysis
%
% organize all the notes I made for comps into topics that can be integrated into an intro if needed
%	
%	May 31:	Complete COG analysis
%	
%	Jun 30:	COG analysis Paper draft completed
%	
%	Jul 31: Add other mol trends to Sub Paper

\underline{Subs Paper Things to Do:}
\begin{itemize}
	\item \sout{ \# of coding and non-coding sites}
	
	\item \sout{\# of subs in each of $\uparrow$}
	
	\item \sout{Look into \strep non-coding issue}
	
	\item Look into \ecol coding issue
	
	\item get dN/dS for coding/non-coding stuff
	 
	\item Or get 1st, 2nd, 3rd codon pos log regs
	
	\item write up coding/non-coding results
	
	\item write up methods for coding/non-coding
	
	\item write methods and results for clustering
	
	\item take out gene expression from this paper
	
	\item write better intro/methods for distribution of subs graphs
	
	\item mol clock for my analysis?
	
	\item write discussion for coding/non-coding
	
	\item GC content? COG? where do these fit?
	
	\item write coding/non-coding into conclusion
	
\end{itemize}

\underline{Gene Expression Paper Things to Do:}
\begin{itemize}
	
	\item \sout{look for more GEO expression data for \smel}
	
	\item \sout{look for more GEO expression data for \strep}
	
	\item \sout{look for more GEO expression data for \bass}
	
	\item find papers about what has been done with gene expression
	
	\item read papers $\uparrow$
	
	\item put notes from $\uparrow$ papers into word doc
	
	\item do same ancestral/phylogenetic analysis that I did in the subs paper 
	
	\item Get numbers for how many different strains and multiples of each strain I have for gene expression
	
	\item format paper and put in stuff that is already written
	
	\item write abstract
	
	\item write intro
	
	\item add stuff from outline to Data section
	
	\item create graphs for expression distribution (no sub data)
	
	\item add \# of genes to expression graphs (top)
	
	\item average gene expression
	
	\item write discussion
	
	\item write conclusion
	
	\item look for more GEO expression data for \ecol
	
	\item add into methods: filters for Hiseq, RT PCR and growth phases for data collection
	
	\item update supplementary figures/file
	
\end{itemize}

\underline{Inversions and Gene Expression Letter Things to Do:}
\begin{itemize}
	\item create latex template for paper
	\item find papers about inversions and expression
	\item read papers $\uparrow$
	\item put notes from papers $\uparrow$ into doc
	\item use large PARSNP alignment to identify inversions
	\item confirm inversions with dot plot
	\item get as much GEO data as possible
	\item write outline for letter
	\item write Abstract
	\item write intro
	\item write methods
	\item compile tables (supplementary)
	\item write results
	\item write discussion
	\item write conclusion 
\end{itemize}




	
\section*{Last Week}

$\checkmark$ look for more GEO expression data for \bass


Last week I finished up running all the coding and non-coding analysis and the results are summarized in the tables below.
You can see that for most replicons, majority of the substitutions are coming from the non-coding regions of the genomes.
The replicons that do not follow this trend is \pb, \pa (which are not chromosomes so this may still be able to be worked into a story) and \strep. 
I am not sure why \strep is showing both coding and non-coding sequences to have higher number of substitutions when moving away from the origin.
Maybe because these taxa are from different strains of \strep, so I am only capturing sequences where all taxa are present, which would likely be important genes, which would be expected to have a positive substitution trend?
Thoughts?

The number of substitutions are higher in the non-coding regions than the coding regions for every replicon except \pb, which is also really weird and interesting. Thoughts?

\ecoli looked like the code was doing something weird and I think it may have to do with my origin and bidirectionality scaling. I am looking into this.
%I think that maybe why we were seeing only negative trends before was because there were more substitutions in the non-coding regions than coding and these non-coding subs were driving the logistic regression to be negative.

I also made a table to summarize the proportion of coding and non-coding sections in each of the genomes (below).

I have also looked at the GEO datasets and attempted to find more for \strep, and \smel. These are summarized also in a table below.
I have only just begun looking for more data in \bass.

%I realized that for \pb I miscalculated the bidirectionality transformation, so I had to fix this and re-run everything.
%It did not change the logistic regression results (seen below).
%However, when I re-did the origin shuffling to see if the placement of the origin changed anything, moving the origin 100kb, 90kb and 80kb to the left made the logistic regression negative (opposite). I have been trying to figure out why this is happening but I am having no luck. I thought maybe it was because these shufflings are now ~700kb away from the terminus, but the actual origin is about the same distance. I am still trying to figure this out but I am not sure what to do or what it means about the robustness of the origin shuffling.

%I looked at the gene expression data for \smel in detail and it appears to look ok.
%When I graph the raw data and plot the regression line it looks like there is no trend for \smel, the points are evenly distributed throughout the genome and there appears to be no increasing or decreasing trend.
%When looking at the other bacteria's raw data, there is clearly a decreasing trend when moving away from the origin.
%Additionally the number of genes and number of replicates are all comparable between \smel and the other bacteria.
%So I think that the reason \smel does not have significant gene expression regressions is because there is simply no trend.

%Last week I was reading an average of 2 articles a day (and will continue to do so until Aug 15). 
%While reading these papers I was researching the increased number of substitutions found near the origin of mitochondria.
%The papers I found on this subject are slightly misleading. The titles suggest that the substitution rate near the Control Region (CR) of mitochondria (which spans the origin on either side) is higher.
%However, after reading the articles I found that the substitution rate is higher than what was previously estimated using phylogenetic methods.
%So, I need to do a bit more research to see if I can find what the substitution rate for the rest of the mt genome is and see if the CR actually is higher.

\section*{This Week}
%aim to tick off 4 tasks a week
I will finish going through the \bass and \ecol GEO data sets to see if there is any more expression data I can grab.

I would like to fix the bidirectionality issue that seems to be happening only with the \ecol coding analysis.

Find papers for the various gene expression papers to see what has already been done in the field and have solid background knowladge.

I would like to create a template in latex for both gene expression papers and add in information that I already have written up.


%In between having this finished I would like to keep looking for gene expression data for \ecoli and have this finished by the end of the week.

\section*{Next Week}

I would like to start figuring out how to get dN/dS for coding and non-coding stuff and/or codon position logistic regression information.

Write out my methods for the coding/non-coding stuff.

Read some of the gene expression papers I will find.

Determine next steps for inversions and gene expression paper.


\newpage
\begin{table}[h]
	\centering
	\resizebox{\textwidth}{!}{%
		\begin{tabular}{lccrr}
			\toprule
			Bacteria and Replicon & \% of Coding Sequences & \% of Non-Coding Sequences & \# of Subs Coding & \# of Subs Non-Coding\\
			\midrule
			\ecol Chromosome & 87.22\% & 12.78\% & 702 & 256423 \\ % 4048616/4641652 & 
			\bass Chromosome & 87.58\% & 12.42\% & 15547 & 287781\\ %3692179/4215606
			\strep Chromosome & 88.02\% & 11.98\% & 1357 & 1200749\\ % 7628849/8667507
			\smel Chromosome & 85.68\% & 14.32\% & 1530 & 5581\\ % 3130925/3654135
			\smel pSymA & 83.34\% & 16.66\% & 3230 & 10343\\ % 1128615/1354226 & 225611/1354226
			\smel pSymB & 88.70\% & 11.30\% & 37419 & 10596\\ % 1493048/1683333
			\bottomrule
		\end{tabular}
		
	}%resizebox
	\caption{\label{tab:cod_non_cod_proportions} Total proportion of coding and non-coding sites in each replicon and the percentage of those sites that have a substitution (multiple substitutions at one site are counted as two substitutions).}
\end{table}




\begin{table}[h]
	\centering
	\resizebox{\textwidth}{!}{%
		\begin{tabular}{lcc}
			\toprule
			Bacteria and Replicon & Coding Sequences & Non-Coding Sequences\\
			\midrule
			\ecol Chromosome & 2.496\e{-5}* & \cc-1.397\e{-7}*** \\
			\bass Chromosome & 1.812\e{-6}*** & \cc-1.439\e{-8}***\\
			\strep Chromosome & 2.984\e{-5}*** & 1.689\e{-8}***\\
			\smel Chromosome & 4.425\e{-6}*** & \cc -1.311\e{-6}***\\
			\smel pSymA & \cc-9.713\e{-7}*** & \cc-1.413\e{-7}***\\
			\smel pSymB & \cc-4.406\e{-7}*** & 5.916\e{-7}***\\
			\bottomrule
		\end{tabular}
		
	}%resizebox
	\caption{\label{tab:cod_non_cod_log_reg} Logistic regression analysis of the number of substitutions along all positions of the genome of the respective bacteria replicons. These genomic positions were split up into the coding and non-coding regions of the genome. Grey coloured boxes indicate a negative logistic regression coefficient estimate. All results are statistically significant. Logistic regression was calculated after the origin of replication was moved to the beginning of the genome and all subsequent positions were scaled around the origin accounting for bidirectionality of replication. All results are marked with significance codes as followed: $<$ 0.001 = `***', 0.001 $<$ 0.01 = `**', 0.01 $<$ 0.05 = `*', 0.05 $<$ 0.1 = `.', $>$ 0.1 = ` '.}
\end{table}



\begin{table}[h]
	\centering
	\resizebox{\textwidth}{!}{%
		\begin{tabular}{lccc}
			\toprule
			Bacteria and Replicon & Coefficient Estimate & Standard Error & P-value \\
			\midrule
			\ecol Chromosome & 2.496\e{-5} & 8.695\e{-6}& 0.0041 \\
			\bass Chromosome & 1.812\e{-6} & 8.913\e{-8} & $<$2\e{-16}\\
			\strep Chromosome & 2.984\e{-5} & 1.858\e{-6} & $<$2\e{-16}\\
			\smel Chromosome & 4.425\e{-6} & 5.155\e{-7} & $<$2\e{-16}\\
			\smel pSymA & -9.713\e{-7} & 3.212\e{-8} & $<$2\e{-16}\\
			\smel pSymB & -4.406\e{-7} & 2.317\e{-8} & $<$2\e{-16}\\
			\bottomrule
		\end{tabular}
		
	}%resizebox
	\caption{\label{tab:cod_log_reg} Logistic regression analysis of the number of substitutions along all coding portions of the genome of the respective bacteria replicons. Grey coloured boxes indicate a negative logistic regression coefficient estimate. All results are statistically significant. Logistic regression was calculated after the origin of replication was moved to the beginning of the genome and all subsequent positions were scaled around the origin accounting for bidirectionality of replication.}
\end{table}

\begin{table}[h]
	\centering
	\resizebox{\textwidth}{!}{%
		\begin{tabular}{lccc}
			\toprule
			Bacteria and Replicon & Coefficient Estimate & Standard Error & P-value \\
			\midrule
			\ecol Chromosome & -1.397\e{-7} & 2.427\e{-9} & $<$ 2\e{-16} \\
			\bass Chromosome & -1.439\e{-8} & 1.569\e{-9} & $<$2\e{-16}\\
			\strep Chromosome & 1.689\e{-8} & 7.235\e{-10} & $<$2\e{-16}\\
			\smel Chromosome & -1.311\e{-6}& 3.393\e{-8}& $<$2\e{-16}\\
			\smel pSymA & -1.413\e{-7} & 3.762\e{-8} & 1.73\e{-4}\\
			\smel pSymB & 5.196\e{-7} & 4.769\e{-8} & $<$2\e{-16}\\
			\bottomrule
		\end{tabular}
		
	}%resizebox
	\caption{\label{tab:non_cod_log_reg} Logistic regression analysis of the number of substitutions along all non-coding portions of the genome of the respective bacteria replicons. Grey coloured boxes indicate a negative logistic regression coefficient estimate. All results are statistically significant. Logistic regression was calculated after the origin of replication was moved to the beginning of the genome and all subsequent positions were scaled around the origin accounting for bidirectionality of replication.}
\end{table}

\begin{table}[h]
	\centering
	\resizebox{0.8\textwidth}{!}{\begin{minipage}{\textwidth}
			\begin{tabular}{lll}
				\toprule
				Bacteria Strain/Species & GEO Accession Number & Date Accessed\\
				\midrule
				\ecol K12 MG1655  & GSE60522 & December 20, 2017\\
				\ecol K12 MG1655 & GSE73673 & December 19, 2017\\
				\ecol K12 MG1655 & GSE85914 & December 19, 2017\\
				\ecol K12 DH10B & GSE98890 & December 19, 2017\\
				\ecol BW25113 & GSE73673 & December 19, 2017 \\
				\ecol BW25113 & GSE85914 & December 19, 2017 \\
				\ecol O157:H7 & GSE46120 & August 28, 2018\\
				\midrule
				\bass 168 & GSE104816 & December 14, 2017\\
				\bass 168 & GSE67058 & December 16, 2017\\
				\bass 168 & GSE93894 & December 15, 2017\\
				\bass 168 & GSE80786 & November 16, 2018\\
				% bass also has quite a few time dependent data sets, or separated by growth phase				
				\midrule
				\scoe A3 & GSE57268 & March 16, 2018\\
				\snat A3 & GSE112559 & November 15, 2018\\
				% strep has LOTS of time dependent data sets, but the other bac do not really have that				
				\midrule
				\smel 1021 Chromosome & GSE69880 & December 12, 2017\\
				%				\smel 1021 Chromosome & GSE121228 & diff phases\\
				\midrule
				\smel 2011 \pa & NC\_020527 (Dr. Finan) & April 4, 2018 \\
				\smel 1021 \pa & GSE69880 & November 15, 18\\
				\midrule
				\smel 2011 \pb & NC\_020560 (Dr. Finan) & April 4, 2018 \\
				\smel 1021 \pb & GSE69880 & November 15, 18\\
				\bottomrule
				
				
			\end{tabular}
			\caption{\label{tab:tableS1} Summary of strains and species found for each gene expression analysis. Gene Expression Omnibus accession numbers and date accessed are provided.}
		\end{minipage}}
	\end{table}	





%\begin{table}[]
%	\centering
%	\resizebox{\textwidth}{!}{%
%		\begin{tabular}{lcccccc}
%			\toprule
%			Position Difference & \ecol Chromosome &\bass Chromosome & \strep Chromosome & \smel Chromosome &\smel pSymA & \smel pSymB\\
%	\midrule
%	1bp & -1.394\e{-7}** & -2.538\e{-8}** & 1.736\e{-8}** & -1.541\e{-6}** & -9.130\e{-7}** & 2.488\e{-7}*** \\
%	10bp & -1.394\e{-7}*** & -2.518\e{-8}*** & -4.484\e{-9}*** & -1.627\e{-6}*** & -9.13\e{-7}*** & 3.487\e{-7}***\\
%	100bp & -1.764\e{-7}*** & -1.417\e{-8}*** & 1.448\e{-8}*** & -1.605\e{-6}*** & -1.166\e{-6}*** & 4.021\e{-7}*** \\
%	1000bp & -1.784\e{-7}*** & -1.417\e{-8}*** & 1.505\e{-8}*** & -1.605\e{-6}*** & -1.153\e{-6}*** & 4.021\e{-7}***\\
%	10000bp & -1.712\e{-7}*** & -3.496\e{-8}*** & 4.790\e{-8}*** & -1.605\e{-6}*** & -3.570\e{-8}* & 3.784\e{-7}*** \\
%	100000bp & -2.061\e{-7}*** & -3.561\e{-8}*** & 4.167\e{-9}*** & -1.605\e{-6}*** & -4.676\e{-7}*** & 3.784\e{-7}***\\
%	1000000bp & 4.229\e{-8}*** & -7.710\e{-9}*** & 6.083\e{-8}*** &-1.605\e{-6}*** & 4.285\e{-6}*** & -8.888\e{-7}*** \\
%	\bottomrule
%		\end{tabular}
%		
%	}%resizebox
%	\caption{\label{tab:clustering} Position clustering analysis. Logistic regression analysis of the number of substitutions along the genome of the respective bacteria replicons to test position differences. Each row denotes different base pair distances that the positions were clustered together as. All results are marked with significance codes as followed: $<$ 0.001 = `***', 0.001 $<$ 0.01 = `**', 0.01 $<$ 0.05 = `*', 0.05 $<$ 0.1 = `.', $>$ 0.1 = ` '. Logistic regression was calculated after the positions in the genome were determined to be the same at each position difference listed in the first column.}
%\end{table}
%
%
%\begin{table}[h]
%	\centering
%	\resizebox{\textwidth}{!}{%
%		\begin{tabular}{lccc}
%			\toprule
%			Bacteria and Replicon & Coefficient Estimate & Standard Error & P-value \\
%			\midrule
%			\cellcolor{black!16}\ecol Chromosome & \cellcolor{black!16}-6.41\e{-5} & \cellcolor{black!16}1.65\e{-5} & \cellcolor{black!16}1.1\e{-4} \\
%			\cellcolor{black!16}\bass Chromosome & \cellcolor{black!16}-9.9\e{-5} & \cellcolor{black!16}2.18\e{-5} & \cellcolor{black!16}6\e{-6} \\
%			\cellcolor{black!16}\strep Chromosome & \cellcolor{black!16}-1.5\e{-6} & \cellcolor{black!16}1.4\e{-7} & \cellcolor{black!16}$<$2\e{-16}\\
%			\smel Chromosome & 3.19\e{-5} & 3.57\e{-5} & 3.7\e{-1}\\
%			\cellcolor{black!16}\smel pSymA & \cellcolor{black!16}-5.36\e{-5} & \cellcolor{black!16}6.34\e{-4} & \cellcolor{black!16}9.33\e{-1} \\
%			\smel pSymB & 5.05\e{-4} & 2.6\e{-4} & 5.3\e{-2}\\
%			\bottomrule
%		\end{tabular}
%		
%	}%resizebox
%	\caption{\label{tab:lr_exp} Linear regression analysis of the median counts per million expression data along the genome of the respective bacteria replicons. Grey coloured boxes indicate statistically significant results at the 0.5 significance level. Linear regression was calculated after the origin of replication was moved to the beginning of the genome and all subsequent positions were scaled around the origin accounting for bidirectionality of replication.}
%\end{table}
%
%\begin{table}[b!]
%	\centering
%	\resizebox{\textwidth}{!}{%
%		\begin{tabular}{lccc}
%			\toprule
%			Bacteria and Replicon & Coefficient Estimate & Standard Error & P-value \\
%			\midrule
%			\cellcolor{black!16}\ecol Chromosome & \cellcolor{black!16}-1.394\e{-7} & \cellcolor{black!16}2.425\e{-9} & \cellcolor{black!16}$<$2\e{-16} \\
%			\cellcolor{black!16}\bass Chromosome & \cellcolor{black!16}-1.265\e{-8} & \cellcolor{black!16}1.562\e{-9} & \cellcolor{black!16}5.430\e{-16} \\
%			\strep Chromosome & 1.736\e{-8} & 7.231\e{-10} & $<$2\e{-16}\\
%			\cellcolor{black!16}\smel Chromosome & \cellcolor{black!16}-1.541\e{-6} & \cellcolor{black!16}3.042\e{-8} & \cellcolor{black!16}$<$2\e{-16}\\
%			\cellcolor{black!16}\smel pSymA & \cellcolor{black!16}-9.130\e{-7} & \cellcolor{black!16}1.975\e{-8} & \cellcolor{black!16}$<$2\e{-16} \\
%			\smel pSymB & 2.488\e{-7} & 1.964\e{-8} & $<$2\e{-16}\\
%			\bottomrule
%		\end{tabular}
%		
%	}%resizebox
%	\caption{\label{tab:tabel2} Logistic regression analysis of the number of substitutions along the genome of the respective bacteria replicons. Grey coloured boxes indicate a negative logistic regression coefficient estimate. All results are statistically significant. Logistic regression was calculated after the origin of replication was moved to the beginning of the genome and all subsequent positions were scaled around the origin accounting for bidirectionality of replication.}
%\end{table}
%
%
%
%\includegraphics[width=\textwidth]{C:/Users/Daniella/Documents/Sinorhizobium2015/Gene_Expression_8Dec17/Gene_Expression/smel_chrom_sub_exp_histogram_bidirectionality_colour_16Mar18}
%
%\includegraphics[width=\textwidth]{C:/Users/Daniella/Documents/Sinorhizobium2015/Gene_Expression_8Dec17/Gene_Expression/pSymA_sub_exp_histogram_bidirectionality_colour_9Apr18}
%
%\includegraphics[width=\textwidth]{C:/Users/Daniella/Documents/Sinorhizobium2015/Gene_Expression_8Dec17/Gene_Expression/pSymB_sub_exp_histogram_bidirectionality_colour_12Apr18}
%
%\includegraphics[width=\textwidth]{C:/Users/Daniella/Documents/Sinorhizobium2015/Gene_Expression_8Dec17/Gene_Expression/bass_chrom_sub_exp_histogram_bidirectionality_colour_5Apr18}
%
%\includegraphics[width=\textwidth]{C:/Users/Daniella/Documents/Sinorhizobium2015/Gene_Expression_8Dec17/Gene_Expression/ecoli_chrom_sub_exp_histogram_bidirectionality_colour_16Mar18}
%
%\includegraphics[width=\textwidth]{C:/Users/Daniella/Documents/Sinorhizobium2015/Gene_Expression_8Dec17/Gene_Expression/strep_chrom_sub_exp_histogram_bidirectionality_colour_19Mar18}
%
%%\begin{table}[h]
%%	\centering
%%	\resizebox{\textwidth}{!}{%
%%		\begin{tabular}{lccc}
%%			\toprule
%%			Bacteria and Replicon & Coefficient Estimate & Standard Error & P-value \\
%%			\midrule
%%			\ecol Chromosome & -1.44\e{-7}& 2.01\e{-9} & $<$2\e{-16}\\
%%			\bass Chromosome & -1.121\e{-7} & 3.41\e{-9} & $<$2\e{-16}\\
%%			\strep Chromosome & 1.24\e{-8} & 7.2\e{-10} & $<$2\e{-16}\\
%%			\smel Chromosome & -1.526\e{-6} & 3.02\e{-8} & $<$2\e{-16}\\
%%			\smel pSymA & -1.058\e{-6}& 2.58\e{-8}& $<$2\e{-16}\\
%%			\smel pSymB & 1.79\e{-7}& 1.84\e{-8}& 1.6\e{-10}\\
%%			\bottomrule
%%		\end{tabular}
%%		
%%	}%resizebox
%%	\caption{\label{tab:tabel2} Logistic regression analysis of the number of substitutions along the genome of the respective bacteria replicons when 10,000bp sections of the genome are re-organized.}
%%\end{table}
%
%
%\begin{table}[]
%	\centering
%	\resizebox{1.2\textwidth}{!}{%
%			\begin{tabular}{lcccccc}
%\toprule
%Origin Location & \ecol Chromosome &\bass Chromosome & \strep Chromosome & \smel Chromosome &\smel pSymA & \smel pSymB\\
%\midrule
%Moved 100kb Left & -1.445\e{-7}*** & 4.374\e{-9}* &  6.909\e{-9}*** &  -1.316\e{-6}*** & -1.058\e{-6}*** & -2.009\e{-7}***\\
%Moved 90kb Left & -1.544\e{-7}*** & -1.036\e{-7}*** & 5.677\e{-9}*** & -1.32\e{-6}*** & -1.246\e{-6}*** &-1.357\e{-7}***\\
%Moved 80kb Left& -1.65\e{-7}*** & -1.072\e{-7}*** & 8.11\e{-9}*** & -1.338\e{-6}*** & -1.398\e{-6}*** & -6.57\e{-8}***\\
%Moved 70kb Left& -1.667\e{-7}*** & -1.102\e{-7}*** & 6.716\e{-9}*** & -1.363\e{-6}*** & -1.405\e{-6}*** & 9.83\e{-8}\\
%Moved 60kb Left& -1.64\e{-7}*** & -1.19\e{-7}*** & 8.7\e{-9}*** & -1.324\e{-6}*** & -1.394\e{-6}*** & 1.129\e{-7}***\\
%Moved 50kb Left& -1.446\e{-7}*** & -1.211\e{-7}*** & 1.045\e{-8}*** & -1.36\e{-6}*** & -1.403\e{-6}*** & 1.521\e{-7}***\\
%Moved 40kb Left& -1.4\e{-7}*** & -1.299\e{-7}*** & 1.214\e{-8}*** & -1.255\e{-6}*** & -1.422\e{-6}*** & 1.543\e{-7}***\\
%Moved 30kb Left& -1.498\e{-7}*** & -1.292\e{-7}*** & 1.24\e{-8}*** & -1.26\e{-6}*** & -1.392\e{-6}*** & 1.63\e{-7}***\\
%Moved 20kb Left& -1.51\e{-7}*** & -1.1\e{-7}*** & 1.395\e{-8}*** & -1.525\e{-6}*** & -1.412\e{-6}*** & 1.603\e{-7}***\\
%Moved 10kb Left& -1.262\e{-7}*** & -2.602\e{-9} & 1.563\e{-8}*** &  -1.599\e{-6}*** &  -9.499\e{-7}*** & 2.973\e{-7}*** \\
%Moved 10kb Right& -1.305\e{-7}*** & -2.045\e{-8}*** & 1.578\e{-8}*** & 1.614\e{-6}*** & -1.026\e{-6}*** & 3.505\e{-7}*** \\
%Moved 20kb Right& -1.454\e{-7}*** & -1.006\e{-7}*** & 1.903\e{-8}*** & -1.634\e{-6}*** & -1.475\e{-6}*** & 1.649\e{-7}***\\
%Moved 30kb Right& -1.548\e{-7}*** & -8.596\e{-8}*** & 2.046\e{-8}*** & -1.698\e{-6}*** & -1.417\e{-6}*** & 1.526\e{-7}***\\
%Moved 40kb Right& -1.632\e{-7}*** & -8.378\e{-8}*** & 2.125\e{-8}*** & -1.719\e{-6}*** & -1.367\e{-6}*** & 1.589\e{-7}***\\
%Moved 50kb Right& -1.856\e{-7}*** & -7.879\e{-8}*** & 1.957\e{-8}*** & -1.735\e{-6}*** & -1.277\e{-6}*** & 1.654\e{-7}***\\
%Moved 60kb Right& -1.91\e{-7}*** & -6.98\e{-8}*** & 1.974\e{-8}*** & -1.788\e{-6}*** & -1.169\e{-6}*** & 1.645\e{-7}***\\
%Moved 70kb Right& -1.892\e{-7}*** & -6.634\e{-8}*** & 1.934\e{-8}*** & -1.854\e{-6}*** & -1.059\e{-6}*** & 1.843\e{-7}***\\
%Moved 80kb Right& -1.879\e{-7}** & -5.814\e{-8}*** & 2.313\e{-8}*** & -1.891\e{-6}*** & -9.07\e{-7}*** & 1.90\e{-7}***\\
%Moved 90kb Right& -1.862\e{-7}*** & -4.314\e{-8}*** & 2.304\e{-8}*** & -1.865\e{-6}*** & -7.171\e{-7}*** & 2.415\e{-7}***\\
%Moved 100kb Right& -1.799\e{-7}*** & -2.597\e{-8}*** &  1.945\e{-8}*** &  -1.525\e{-6}*** & -6.572\e{-7}*** & 3.095\e{-7}***\\
%\bottomrule
%\end{tabular}
%		
%	}%resizebox
%	\caption{\label{tab:tabel2} Logistic regression analysis of the number of substitutions along the genome of the respective bacteria replicons. All results are marked with significance codes as followed: $<$ 0.001 = `***', 0.001 $<$ 0.01 = `**', 0.01 $<$ 0.05 = `*', 0.05 $<$ 0.1 = `.', $>$ 0.1 = ` '. Logistic regression was calculated after the origin of replication was moved to the beginning of the genome and all subsequent positions were scaled around the origin accounting for bidirectionality of replication.}
%\end{table}
%
%%\begin{table}[h]
%%	\resizebox{\textwidth}{!}{%
%%		\begin{tabular}{lccc}
%%			\toprule
%%			Bacteria Replicon & \multicolumn{1}{p{3cm}}{\centering \% of Total LCBs \\ with Identical Tree} & 
%%			\multicolumn{1}{p{3cm}}{\centering \% of Total LCBs \\ with Not Identical Tree} & 
%%			\multicolumn{1}{p{3cm}}{\centering \% of Total Alignment Discarded} \\  
%%			\midrule
%%			\ecol Chromosome & 81.58\% & 18.42\% & 25.18\%\\
%%			\bass Chromosome & 83.33\% & 16.67\% & 19.37\%\\
%%			\strep Chromosome & 96.53\% & 3.47\% & 12.42\%\\
%%			\smel Chromosome & 81.82\% & 18.18\% & 25.42\%\\
%%			\smel \pa & 100\% & 0\% & 0\%\\
%%			\smel \pb & 100\% & 0\% & 0\%\\
%%			\bottomrule
%%		\end{tabular}
%%	}%resize box
%%\end{table}
%%
%% 
%
%%
%%\clearpage
%%
%%\includegraphics[width=\textwidth]{C:/Users/Daniella/Documents/Sinorhizobium2015/Figs/Bidirectionality_outliers_coloured_9Jun17/bass_chrom_change_histogram_bidirectionality_colour_6Nov17.pdf}
%%
%%\includegraphics[width=\textwidth]{C:/Users/Daniella/Documents/Sinorhizobium2015/Figs/Bidirectionality_outliers_coloured_9Jun17/streo_chrom_change_histogram_bidirectionality_colour_6Nov17.pdf}
%%
%%\includegraphics[width=\textwidth]{C:/Users/Daniella/Documents/Sinorhizobium2015/Figs/Bidirectionality_outliers_coloured_9Jun17/chrom_change_histogram_bidirectionality_colour_24Nov17.pdf}
%%
%%\includegraphics[width=\textwidth]{C:/Users/Daniella/Documents/Sinorhizobium2015/Papers/Substitutions_paper/Substitutions_paper/Figs/pSymA_change_histogram_bidirectionality_colour_6Oct17.pdf}
%%
%%\includegraphics[width=\textwidth]{C:/Users/Daniella/Documents/Sinorhizobium2015/Papers/Substitutions_paper/Substitutions_paper/Figs/pSymB_change_histogram_bidirectionality_colour_10Oct17.pdf}
%%
%%
\end{document}
